<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html lang="en">
<head>
<link rel="stylesheet" href="../stylesheets/styles.css">
<link rel="stylesheet" href="../stylesheets/pygment_trac.css">
<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
   
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="Content-Style-Type" content="text/css">
<title>Gota Morota - Research projects</title>
</head>

<body>
<p class="menublock"><a href="../index.html">Back</a></p>



<h1>Key research projects</h1>



<h2>1. Statistical quantitative genetic analysis of high-throughput phenomics data</h2>
<p>
  We develop and apply statistical methods that can be used to handle data generated from
high-throughput phenotyping or precision agriculture. 

  <ul>
     
    <li>
      Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. 2019. Leveraging breeding values obtained from random regression models for genetic inference of longitudinal traits. The Plant Genome. Early view. 
      [<a href="https://doi.org/10.3835/plantgenome2018.10.0075">DOI</a>]
      [<a href="https://doi.org/10.1101/435685">bioRxiv</a>]
    </li>


     <li>
      Hussain W, Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. 2018. ShinyAIM: Shiny-based application of interactive Manhattan plots for longitudinal genome-wide association studies. Plant Direct. 2:e00091.
      [<a href="https://doi.org/10.1002/pld3.91">DOI</a>]
      [<a href="https://doi.org/10.1101/383026">bioRxiv</a>]
    </li>


      <li> Campbell MT, Walia H, and  <strong><u>Morota G</u></strong>. 2018. Utilizing random regression models for genomic prediction of a longitudinal trait derived from high-throughput phenotyping. Plant Direct. 2:e00080.
     [<a href="https://doi.org/10.1002/pld3.80">DOI</a>]
      [<a href="https://doi.org/10.1101/319897">bioRxiv</a>]
    </li>

     <li><strong><u>Morota G</u></strong>, Ventura RV, Silva FF, Koyama M, and Fernando SC. 2018. Machine learning and data mining advance predictive big data analysis in precision animal agriculture. Journal of Animal Science. 96:1540â€“1550.
      [<a href="http://dx.doi.org/10.1093/jas/sky014">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29385611">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/29385611">EuropePMC</a>]
    </li>



    </ul>
</p>


<h2>2. Network analysis</h2>
<p>
  Network-based quantitative genetics offers perspective for new approaches. 
  <ul>

    <li>
      Momen M, Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. Harnessing phenotypic networks and structural equation models to improve genome-wide association analysis. bioRxiv.
	[<a href="https://doi.org/10.1101/553008">bioRxiv</a>]
    </li>

    
     <li>
      Yu H, Campbell MT, Zhang Q, Walia H, and <strong><u>Morota G</u></strong>. Genomic Bayesian confirmatory factor analysis and Bayesian network to characterize a wide spectrum of rice phenotypes. bioRxiv.
	[<a href="https://doi.org/10.1101/435792">bioRxiv</a>]
    </li>

    <li>
      Momen M, Mehrgardi AA, Roudbar MA, Kranis A, Pinto RM, Valente BD, <strong><u>Morota G</u></strong>, Rosa GJM, and Gianola D. 2018. Including phenotypic causal networks in genome-wide association studies using mixed effects structural equation models. Frontiers in Genetics. 9:455.
      [<a href="https://doi.org/10.3389/fgene.2018.00455">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/30356716">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/30356716">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/251421">bioRxiv</a>]
    </li>   
    


      <li>Valente BD, <strong><u>Morota G</u></strong>, Penagaricano F, Gianola D, Weigel KA, and Rosa GJM. 2015. The causal meaning of genomic predictors and how it affects  construction and comparison of genome-enabled selection models. Genetics. 200:483-494. 
    [<a href="http://dx.doi.org/10.1534/genetics.114.169490">DOI</a>]
    [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25908318">PubMed</a>]
    [<a href="http://europepmc.org/abstract/MED/25908318">EuropePMC</a>]
    [<a href="http://dx.doi.org/10.1101/001511">bioRxiv</a>] 
    [<a href="http://arxiv.org/abs/1401.1165">arXiv</a>] 
  </li>


   <li><strong><u>Morota G</u></strong> and Gianola D. 2013. 
Evaluation of linkage disequilibrium in wheat with an L1 regularized sparse Markov network. Theoretical and Applied Genetics. 126:1991-2002. 
  [<a href="http://dx.doi.org/10.1007/s00122-013-2112-y">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23661079">PubMed</a>] 
  [<a href="http://europepmc.org/abstract/MED/23661079">EuropePMC</a>]
</li>

<li><strong><u>Morota G</u></strong>, Valente BD, Rosa GJM,  Weigel KA, and Gianola D. 2012. 
 An assessment of linkage disequilibrium in Holstein cattle using a Bayesian network. 
  Journal of Animal Breeding and Genetics. 129:474-487. 
  [<a href="http://dx.doi.org/10.1111/jbg.12002">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23148973">PubMed</a>] 
  [<a href="http://europepmc.org/abstract/MED/23148973">EuropePMC</a>]
  [<a href="http://onlinelibrary.wiley.com/journal/10.1111/%28ISSN%291439-0388/homepage/genomic_selection_virtual_issue.htm">Genomic Selection Virtual Issue</a>]
</li>

      
      
    </ul>
</p>


<h2>3. Genomic connectedness</h2>
<p>
  We showed that genetic connectedness can be used in the context of genomic prediction. 
  <ul>

     <li> Momen M and <strong><u>Morota G</u></strong>. 2018. Quantifying genomic connectedness and prediction accuracy from additive and non-additive gene actions. Genetics Selection Evolution. 50:45. 
      [<a href="https://doi.org/10.1186/s12711-018-0415-9">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/30223766">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/30223766">EuropePMC</a>]
   </li>

     
    <li>Yu H, Spangler ML, Lewis RM, and <strong><u>Morota G</u></strong>. 2018. Do stronger measures of genomic connectedness enhance prediction accuracies across management units? Journal of Animal Science. 96:4490-4500.
      [<a href="https://doi.org/10.1093/jas/sky316">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/30165381">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/30165381">EuropePMC</a>]
  </li>


     <li>Yu H, Spangler ML, Lewis RM, and <strong><u>Morota G</u></strong>. 2017. Genomic relatedness strengthens genetic connectedness across management units.  G3: Genes, Genomes, Genetics. 10:3543-3556.
      [<a href="https://doi.org/10.1534/g3.117.300151">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28860185">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/28860185">EuropePMC</a>]
      [<a href="https://dx.doi.org/10.1101/130138">bioRxiv</a>]
    </li>

       

    </ul>
</p>



<h2>4. Medical Subject Headings (MeSH) analysis</h2>
<p>
  We pioneered MeSH analysis in agricultural species.
  <ul>

    <li>
      Abdollahi-Arpanahi R, <strong><u>Morota G</u></strong>, and Penagaricano F. 2017. Predicting bull fertility using genomic data and biological information. Journal of Dairy Science. 100:9656-9666. 
      [<a href="https://doi.org/10.3168/jds.2017-13288">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28987577">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/28987577">EuropePMC</a>]
    </li>


      <li>Beissinger TM and <strong><u>Morota G</u></strong>. 2017. Medical subject heading (MeSH) annotations illuminate maize genetics and evolution. Plant Methods. 13:8. 
      [<a href="http://dx.doi.org/10.1186/s13007-017-0159-5">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28250803">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/28250803">EuropePMC</a>]
      [<a href="http://dx.doi.org/10.1101/048132">bioRxiv</a>]
    </li>

       <li><strong><u>Morota G</u></strong>, Beissinger TM, and Penagaricano F. 2016.  MeSH-informed enrichment analysis and MeSH-guided semantic similarity among functional terms and gene products in chicken. G3: Genes, Genomes, Genetics. 6:2447-2453. 
      [<a href="http://dx.doi.org/10.1534/g3.116.031096">DOI</a>]
      [<a href="http://www.ncbi.nlm.nih.gov/pubmed/27261003">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/27261003">EuropePMC</a>]
      [<a href="http://dx.doi.org/10.1101/034975">bioRxiv</a>]
    </li>

      <li><strong><u>Morota G</u></strong>, Penagaricano F, Petersen JL, Ciobanu DC, 2015. Tsuyuzaki K, and Nikaido I. An application of MeSH enrichment analysis in livestock. Animal Genetics. 46:381-387.
    [<a href="http://dx.doi.org/10.1111/age.12307">DOI</a>]
    [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26036323">PubMed</a>]
    [<a href="http://europepmc.org/abstract/MED/26036323">EuropePMC</a>]
</li>

      <li>Tsuyuzaki K, <strong><u>Morota G</u></strong>, Ishii M, Nakazato T, Miyazaki S, and Nikaido I. 
  2015. MeSH ORA framework: R/Bioconductor packages to support MeSH over-representation analysis.
  BMC Bioinformatics. 16:45. 
  [<a href="http://dx.doi.org/10.1186/s12859-015-0453-z">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25887539">PubMed</a>]
  [<a href="http://europepmc.org/abstract/PMC/PMC4343279">EuropePMC</a>]
</li>

    </ul>
</p>






<h2>5. Software development</h2>
<p>
  We are interested in developing modern interactive visualization tools in agriculture science. 
  
  <ul>
     
    <li>
      Hussain W, Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. 2018. ShinyAIM: Shiny-based application of interactive Manhattan plots for longitudinal genome-wide association studies. Plant Direct. 2:e00091.
      [<a href="https://doi.org/10.1002/pld3.91">DOI</a>]
      [<a href="https://doi.org/10.1101/383026">bioRxiv</a>]
    </li>


      <li><strong><u>Morota G</u></strong>. ShinyGPAS: 2017. Interactive genomic prediction accuracy simulator based on deterministic formulas. Genetics Selection Evolution. 49:91. 
      [<a href="https://doi.org/10.1186/s12711-017-0368-4">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29262775">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/29262775">EuropePMC</a>]
      [<a href="https://dx.doi.org/10.1101/164772">bioRxiv</a>]
    </li>
   

    </ul>
</p>







</body>
</html>
