\documentclass[margin,line,10pt]{res}

\usepackage{verbatim}
\usepackage{hyperref}
\usepackage{color}
\usepackage{multicol}
\usepackage{url}
\newcommand\doilink[1]{\href{http://dx.doi.org/#1}{#1}}
\newcommand\doi[1]{doi:\doilink{#1}}


\oddsidemargin -.5in
\evensidemargin -.5in
\textwidth=6.0in
\itemsep=0in
\parsep=0in


\newenvironment{list1}{
  \begin{list}{\ding{113}}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in} 
      \setlength{\leftmargin}{0.17in}}}{\end{list}}

\newenvironment{list2}{
  \begin{list}{$\bullet$}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in} 
      \setlength{\leftmargin}{0.2in}}}{\end{list}}




\begin{document}

\name{Gota Morota \hspace{12.2cm} January 2016 \vspace*{.1in}}

\begin{resume}
\section{\sc Contact Information}
\vspace{.05in}
\begin{tabular}{@{}p{2in}p{4in}}
A218f Animal Science Building   & \hspace{2.5cm} {\it E-mail:}  morota@unl.edu\\       
Department of Animal Science  & \hspace{2.5cm} {\it Phone:} (402)-472-6031\\     
University of Nebraska-Lincoln  & \hspace{2.5cm} {\it WWW:} \textcolor{blue}{\href{www.morotalab.org}{morotalab.org}  }\\
Lincoln, NE 68583-0908 USA & \\
\end{tabular}


\vspace{0.4cm}
\section{\sc Research Interests}
I am a quantitative geneticist interested in incorporating statistics, machine learning, 
and bioinformatics to the study of animal genetics in the omics era. 
The core line of my research is connecting the quantitative genetics and animal breeding theories to currently available molecular information. 
I am particularly interested in statistical methods for prediction of complex traits using whole-genome molecular markers. 
%In particular, I am working on integrating the different types of omics data for the prediction of complex traits using a broad range of agricultural species. 


\vspace{0.4cm}
\section{\sc Education}

{\bf University of Wisconsin-Madison}, Madison, Wisconsin USA\\
\vspace*{-.1in}
\begin{list1}
\item[] Ph.D., Animal Sciences, May 2014
\begin{list2}
\vspace*{.05in}
\item Dissertation: ``Whole-genome Prediction of Complex Traits Using Kernel Methods." 
\item Advisor: Prof. Dr. Daniel Gianola 
\item Committee: Drs. Corinne D. Engelman, Guilherme J. M. Rosa, Grace Wahba and Kent A. Weigel
\end{list2}
\vspace*{.05in}
\end{list1}





{\bf University of Wisconsin-Madison}, Madison, Wisconsin USA\\
\vspace*{-.1in}
\begin{list1}
\item[] M.S., Dairy Science, December 2011
\begin{list2}
\vspace*{.05in}
\item Thesis: ``Application of Bayesian and Sparse Network Models for Assessing Linkage Disequilibrium in Animals and Plants." 
\item Advisor: Prof. Dr. Daniel Gianola 
\item Committee: Drs. Guilherme J. M. Rosa and Kent A. Weigel
\end{list2}
\vspace*{.05in}
\end{list1}


{\bf Obihiro University of Agriculture and Veterinary Medicine}, Obihiro, Hokkaido Japan\\
\vspace*{-.1in}
\begin{list1}
\item[] B.S., Agricultural Science,  March 2008
\begin{list2}
\vspace*{.05in}
\item Thesis:  ``Genetic Analysis of Threshold Traits." 
\item Advisor: Prof. Dr. Mitsuyoshi Suzuki
\end{list2}
\end{list1}




\vspace{0.4cm}
\section{\sc Professional   Positions}
Department of Animal Science\\
{\bf University of Nebraska-Lincoln}, Lincoln, Nebraska USA
\vspace{-.4cm}

Assistant Professor (70\% Research \& 30\% Teaching)   \hfill {\bf 08/2014 - Present}\\



\begin{comment}
\section{\sc Professional   Appointments}
Affiliate Faculty Member \\
Department of xxx\\
{\bf University of Nebraska-Lincoln}, Lincoln, Nebraska USA
\end{comment}


\vspace{0.4cm}
\section{\sc Work \phantom{1cm} Experience}
Department of Animal Sciences\\
{\bf University of Wisconsin-Madison}, Madison, Wisconsin USA


\vspace{-.4cm}

Graduate Research Assistant   \hfill {\bf 06/2011 - 05/2014}\\
\vspace{-.4cm}

Animal Genetics Research \& Development  Group \\
{\bf Zoetis, Inc.}, Kalamazoo, Michigan USA

\vspace{-.4cm}

Quantitative Geneticist (student internship)   \hfill {\bf 06/2013 - 11/2013}\\








\vspace{0.5cm}
\section{\sc Professional society memberships}
\begin{list2}
\item  The International Society for Animal Genetics. 2015 - Present
\vspace{0.3cm}
\item The American Society of Animal Science. 2014 - Present
\vspace{0.3cm}
\item The Japanese Society of Animal Breeding and Genetics. 2013 - Present 
\vspace{0.3cm}
\item The International Biometric Society (ENAR). 2012 - Present
\end{list2}



\vspace{0.5cm}
\section{\sc Editorial Activities}
\vspace{.1cm}
Reviewed for: Animal Genetics (1), Animal Production Science (1), BMC Genetics (2), Computers and Electronics in Agriculture (1), DNA Research (1), G3: Genes, Genomes, Genetics (1), Genetics (2), Genetics Selection Evolution (2), Journal of Animal Breeding and Genetics (2), Journal of Animal Science and Biotechnology (1), PLoS ONE (1), Poultry Science (1), Scientia Agricola (2) 
\vspace{.1cm}

Number of reviewed papers:
2012 (1), 2013 (1), 2014 (6), 2015 (10) 
%2015 (2)
%Scientia Agricola
%Genetics Selection Evolution



\vspace{0.5cm}
\section{\sc Preprints}
\begin{list1}
\item [{\bf 15}.]  \textbf{\underline{Morota G}}, Beissinger TM, and Pe\~{n}agaricano F. MeSH annotation of the chicken genome: MeSH-informed enrichment analysis and MeSH-guided semantic similarity among functional terms and gene products. bioRxiv. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1101/034975}{10.1101/034975}}.  
\end{list1}


\vspace{0.5cm}
\section{\sc Peer reviewed journal papers}
\vspace{1cm}

\section{\sc 2016}

\begin{list1}

  \item [{\bf 14}.] Abdollahi-Arpanahi R, {\bf \underline{Morota G}}, Valente BD, Kranis A, Rosa GJM, and Gianola D. 
  2016. Differential contribution of genomic regions to marked genetic variation and prediction of quantitative traits in broiler chickens. \emph{Genetics Selection Evolution}. \text{48}:10. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1186/s12711-016-0187-z}{10.1186/s12711-016-0187-z}}.  

\end{list1}
  
\section{\sc 2015}

\begin{list1}

\item [{\bf 13}.]  Hu Y, {\bf \underline{Morota G}}, Rosa GJM, and Gianola D. 2015. Prediction of plant height in Arabidopsis thaliana from DNA methylation data. \emph{Genetics}. \textbf{201}:779-793. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1534/genetics.115.177204}{10.1534/genetics.115.177204}}.  

\vspace{0.5cm}

\item [{\bf 12}.]  Valente BD, {\bf \underline{Morota G}}, Pe\~{n}agaricano F, Gianola D, Weigel KA, and Rosa GJM. 2015. The causal meaning of genomic predictors and how it affects  construction and comparison of genome-enabled selection models. \emph{Genetics}. \textbf{200}:483-494. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1534/genetics.114.169490}{10.1534/genetics.114.169490}}.  

\vspace{0.5cm}

\item [{\bf 11}.] {\bf \underline{Morota G}}, Pe\~{n}agaricano F, Petersen JL, Ciobanu DC, Tsuyuzaki K, and Nikaido I. 2015. An application of MeSH enrichment analysis in livestock. \emph{Animal Genetics}. \textbf{46}:381-387. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1111/age.12307}{10.1111/age.12307}}.  

\vspace{0.5cm}

\item [{\bf 10}.] Abdollahi-Arpanahi R, {\bf \underline{Morota G}}, Valente BD, Kranis A, Rosa GJM, and Gianola D. 
  (2015) Assessment of bagging GBLUP for whole-genome prediction of broiler chicken traits. \emph{Journal of Animal Breeding and Genetics}. \textbf{132}:218-228. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1111/jbg.12131}{10.1111/jbg.12131}}.  

\vspace{0.5cm}

\item [{\bf 9}.]  Tsuyuzaki K, {\bf \underline{Morota G}}, Ishii M, Nakazato T, Miyazaki S, and Nikaido I. 2015. 
  MeSH ORA framework: R/Bioconductor packages to support MeSH over-representation analysis. \emph{BMC Bioinformatics}. {\bf 16}:45. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1186/s12859-015-0453-z}{10.1186/s12859-015-0453-z}}. 

\end{list1}


\section{\sc 2014}

\begin{list1}


\item [{\bf 8}.]  {\bf \underline{Morota G}} and Gianola D. 2014. 
  Kernel-based whole-genome prediction of complex traits: a review. \emph{Frontiers in Genetics}. {\bf 5}:363. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.3389/fgene.2014.00363}{10.3389/fgene.2014.00363}}. 

\vspace{0.5cm}

\item [{\bf 7}.]  {\bf \underline{Morota G}}, Boddhireddy P, Vukasinovic N, Gianola D, and DeNise S.  2014. Kernel-based variance components estimation and  whole-genome prediction of pre-corrected phenotypes and progeny tests for dairy cow health traits. \emph{Frontiers in Genetics}. {\bf 5}:56. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.3389/fgene.2014.00056}{10.3389/fgene.2014.00056}}. 

\vspace{0.5cm}

\item [{\bf 6}.]  {\bf \underline{Morota G}}, Abdollahi-Arpanahi R, Kranis A, and Gianola D. 2014.   
     Genome-enabled prediction of broiler traits in chickens using genomic annotation. \emph{BMC Genomics}. {\bf 15}:109. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1186/1471-2164-15-109}{10.1186/1471-2164-15-109}} . 
    

\vspace{0.5cm}

\item [{\bf 5}.]  Abdollahi-Arpanahi R, Pakdel A, Nejati-Javaremi A, Moradi-Shahrbabak M, 
     {\bf \underline{Morota G}}, Valente BD, Kranis A, Rosa GJM, and Gianola D. 2014.
     Dissection of additive genetic variability for quantitative traits in chickens using SNP markers. \emph{Journal of Animal Breeding and Genetics}. \textbf{131}:183-193. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1111/jbg.12079}{10.1111/jbg.12079}}. 

\vspace{0.5cm}

\item [{\bf 4}.]  Abdollahi-Arpanahi R,  Nejati-Javaremi A,  Pakdel A, Moradi-Shahrbabak M, 
     {\bf \underline{Morota G}}, Valente BD, Kranis A, Rosa GJM, and Gianola D. 2014.   
     Effect of allele frequencies, effect sizes and number of markers on prediction of quantitative 
     traits in chickens. \emph{Journal of Animal Breeding and Genetics}. \textbf{131}:123-133. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1111/jbg.12075}{10.1111/jbg.12075}}. 
\end{list1}

\section{\sc 2013}


\begin{list1}
\item [{\bf 3}.]  {\bf \underline{Morota G}}, Koyama M, Rosa GJM, Weigel KA, and Gianola D. 2013.
     Predicting complex traits using a diffusion kernel on genetic markers with an application to dairy cattle and wheat data. \emph{Genetics Selection Evolution}. {\bf 45}:17. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1186/1297-9686-45-17}{10.1186/1297-9686-45-17}}.  

\vspace{0.5cm}

\item [{\bf 2}.] {\bf \underline{Morota G}} and Gianola D. 2013.  Evaluation of linkage disequilibrium in wheat with an L1 regularized sparse Markov network.
  \emph{Theoretical and Applied Genetics}. {\bf 126}:1991-2002. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1007/s00122-013-2112-y}{10.1007/s00122-013-2112-y}}. 
\end{list1}


\section{\sc 2012}
\begin{list1} 
\item [{\bf 1}.] {\bf \underline{Morota G}}, Valente BD, Rosa GJM, Weigel KA, and Gianola D. 2012.  
An assessment of linkage disequilibrium in Holstein cattle using a Bayesian network. \emph{Journal of Animal Breeding and Genetics}. {\bf 129}:474-487. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1111/jbg.12002}{10.1111/jbg.12002}}. 
\end{list1}



\vspace{1.0cm}
\section{\sc Papers in Proceedings}
\vspace{0.9cm}
\section{\sc 2014}
\begin{list1}
\item [\bf{3}.] Gianola D, {\bf \underline{Morota G}}, and Crossa J. 2014. 
Genome-enabled Prediction of Complex Traits with Kernel Methods: What Have We Learned? In: \emph{Proceedings, 10th World Congress of Genetics Applied to Livestock Production}. August 17-22, Vancouver, BC, Canada. 
\textcolor{blue}{\href{http://www.morotalab.org/publications/pdf/gianola2014WCGALP.pdf}{PDF} } 


\vspace{0.5cm}

\item [\bf{2}.] Valente BD, {\bf \underline{Morota G}}, Rosa GJM, Gianola D, and  Weigel KA 2014. 
Causal meaning of genomic predictors: Implication on genome-enabled selection modeling. In: \emph{Proceedings, 10th World Congress of Genetics Applied to Livestock Production}. August 17-22, Vancouver, BC, Canada. 
\textcolor{blue}{\href{http://www.morotalab.org/publications/pdf/valente2014WCGALP.pdf}{PDF} } 
\end{list1}

\section{\sc 2011}
\begin{list1}
\item [\bf{1}.] Bueno Filho JS*, {\bf \underline{Morota G}}*, Tran Q, Maenner MJ, Vera-Cala LM, Engelman CD, and Meyers KJ. 2011. Analysis of human mini-exome sequencing data from Genetic \& Analysis Workshop 17 using a  Bayesian hierarchical mixture model. In: \emph{BMC Proceedings}. {\bf 5}(Suppl 9):S93. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1186/1753-6561-5-S9-S93}{10.1186/1753-6561-5-S9-S93}}. *equal contribution  
\end{list1}






%\section{\sc Articles and Editorials not peer reviewed}


\vspace{0.5cm}
\section{\sc Invited Presentations}
\vspace{0.5cm}
My PDF presentation slides are available on \textcolor{blue}{\href{http://www.slideshare.net/chikudaisei/}{SlideShare}} (http://www.slideshare.net/chikudaisei/). \\


\section{\sc 2016}
\begin{list2}
  
  \item [{\bf 6}.] MeSH annotation of the chicken genome. The Plant and Animal Genome XXIV Conference. Town and Country Hotel, San Diego, CA. January 9-13. 
  
\end{list2}

\section{\sc 2015}
\begin{list1}
\item [{\bf 5}.] Inferring the impact of population stratification on genomic heritability using a reparameterized genomic best linear unbiased prediction model. Statistics Seminars. Department of Statistics, University of Nebraska-Lincoln. September 23. 

  \vspace{0.5cm}

\item [{\bf 4}.] Quantitative genetics in the functional genomics era. Animal Breeding \& Genetics Seminars. Department of Animal Science, Iowa State University. March 3. 
\end{list1}

\section{\sc 2014}
\begin{list1}
\item [{\bf 3}.] Quantitative genetics in the functional genomics era. Special Seminar. PIC, Inc., Hendersonville, TN.  November 12. 
\end{list1}

\section{\sc 2013}
\begin{list1}
\item [{\bf 2}.] Whole-genome prediction of complex traits using kernel methods. Department of Animal Science, University of Nebraska-Lincoln. December 19. 
\end{list1}

\section{\sc 2011}
\begin{list1}
\item [{\bf 1}.] Obihiro GCOE Animal Global Health Seminars.  Obihiro University of Agriculture and  Veterinary Medicine. January 7. 
\end{list1}  



\vspace{0.5cm}
%\section{\sc Contributed Abstracts}
\section{\sc Contributed Presentations}
\vspace{1.0cm}

\section{\sc 2015}
\begin{list2}

  \item [{\bf 6}.] Quantitative genetics in the functional genomics era. Special Seminar. The National Institute of Agrobiological Sciences, Tsukuba, Japan. November 12. 

  \vspace{0.5cm}

\item [{\bf 5}.] Quantitative genetics in the functional genomics era. Special Seminar. Laboratory of Biometry and Bioinformatics, The University of Tokyo. November 6. 

  \vspace{0.5cm}
  
\item [{\bf 4}.] The impact of population stratification on genomic heritability. NCERA-225 Meeting. North Dakota State University. October 22-23. 

  \vspace{0.5cm}

\item [{\bf 3}.] An application of MeSH enrichment analysis in livestock. ADSA-ASAS Joint Annual Meeting. Rosen Shingle Creek, Orlando FL. July 12-16.  

  \vspace{0.5cm}
  
\item [{\bf 2}.] Prediction of complex quantitative traits using functional annotations and bootstrap aggregating. Special Seminar. National Livestock Breeding Center, Shirakawa, Japan. January 10. 
\end{list2}  

\section{\sc 2012}
\begin{list2}
\item  [{\bf 1}.] Application of Bayesian and Sparse Network Models for Assessing Linkage Disequilibrium in Animals and Plants. 26th International Biometric Conference. Kobe International Conference Center, Kobe Japan. August 26-31.
  \textcolor{blue}{\href{http://secretariat.ne.jp/ibc2012/30Aug.html\#aug-30-14:00-Contributed36}{http://secretariat.ne.jp/ibc2012/30Aug.html\#aug-30-14:00-Contributed36. } }   {\bf $\star$Second Oral Prize Winners}.
\end{list2}




\vspace{0.5cm}
\section{ \sc Posters }
\vspace{0.5cm}
\section{\sc 2015}
\begin{list2}

\item Population stratification contribution to genomic heritability. Probabilistic Modeling in Genomics. Cold Spring Harbor Laboratory, NY. October 14 - 17. 

  \vspace{0.5cm}
  
\item Estimating genomic heritability in the presence of population stratification. NGS Field 4th Meeting. Tsukuba International Congress Center, Tsukuba, Japan. July 1-3. 
\end{list2}

\section{\sc 2013}
\begin{list2}
\item   MeSHR: R/Bioconductor package for finding statistically overrepresented  MeSH terms in a set of genes. Annual Bioconductor Conference BioC 2013. July 18-19, Seattle, WA. \\ \textcolor{blue}{\href{https://secure.bioconductor.org/BioC2013/posters.php\#7}{https://secure.bioconductor.org/BioC2013/posters.php\#8. } }  

\vspace{0.5cm}

\item Predicting complex traits using a diffusion kernel on genetic markers with an application to dairy cattle and wheat data. Annual Bioconductor Conference BioC 2013. July 18-19, Seattle, WA. \textcolor{blue}{\href{https://secure.bioconductor.org/BioC2013/posters.php\#7}{https://secure.bioconductor.org/BioC2013/posters.php\#7. } }  
\end{list2}








%\section{SCIENCE OUTREACH & VOLUNTEER}



\vspace{0.5cm}
\section{\sc Intramural Seminars}
\vspace{1cm}
\section{\sc 2015}
\begin{list2}
\item  Animal Breeding \& Genetics Seminars. Department of Animal Science, University of Nebraska-Lincoln. September 15. 
\end{list2}  
\section{\sc 2014}
\begin{list2}
\item  Animal Breeding \& Genetics Seminars. Department of Animal Science, University of Nebraska-Lincoln. September 18. 
\vspace{0.25cm}
\item  Ph.D. Thesis Defense. Department of Animal Sciences, University of Wisconsin-Madison. May 12. 
\vspace{0.25cm}
\item  Dairy Science Graduate Seminars. Department of Dairy Science, University of Wisconsin-Madison. February 14. 
\end{list2}  
\section{\sc 2013}
\begin{list2}
\item  Special Seminar.  Zoetis, Inc., Kalamazoo, MI. August 8. 
\end{list2}  
\section{\sc 2012}
\begin{list2}
 \item Animal Breeding \& Genomics Seminars.  Department of Animal Sciences, University of Wisconsin-Madison. April 10.
\end{list2}  
\section{\sc 2011}
\begin{list2}
\item  Master's Thesis Defense.  Department of Dairy Science, University of Wisconsin-Madison. December 5. 
\end{list2}  
\section{\sc 2010}
\begin{list2}
\item Animal Breeding \& Genomics Seminars.  Department of Dairy Science, University of Wisconsin-Madison. November 30.
\vspace{0.25cm}
\item Animal Breeding \& Genomics Seminars.  Department of Dairy Science, University of Wisconsin-Madison. March 23. 
\end{list2}
\section{\sc 2008}
\begin{list2}
\item  Animal Breeding \& Genomics Seminars.  Department of Dairy Science, University of Wisconsin-Madison. November 25. 
\end{list2}




\vspace{0.5cm}
\section{\sc Teaching}
%jointly with Drs.
{\bf University of Nebraska-Lincoln}, Lincoln, Nebraska USA 
\vspace{.01pt}

Instructor \\
ASCI 896 Statistical Genomics    \hfill {\bf Spring, 2016}
\vspace{.01pt}

Guest Instructor \\
ASCI 432/832 Genome Analysis - April 16    \hfill {\bf Spring, 2015}
\vspace{.01pt}

{\bf University of Wisconsin-Madison}, Madison, Wisconsin USA 
\vspace{.01pt}

Teaching Assistant \\
ANSCI/DYSCI 363: Principles of Animal Breeding    \hfill {\bf Spring, 2011}


\vspace{.01pt}
Teaching Assistant \\
ANSCI/DYSCI 361: Introduction to Animal and Veterinary Genetics    \hfill {\bf Spring, 2011} 




\vspace{0.5cm}
\section{\sc Research Support}
\begin{flushleft}
\hspace{0.2cm} \underline{Internal Funding}
\end{flushleft}
\begin{list2}
\item Layman Seed Award - \$9,910.00  \hfill UNL\\
PI: Gota Morota   \hfill \textbf{June 1, 2015 - May 31, 2016}\\
Proposal: Cracking the Blackbox of Whole-genome Prediction: Genome Partitioning of Predictive Ability \\
Role: Principal Investigator \\ 
\end{list2}



\vspace{0.5cm}
\section{\sc Service Activities} 
%International and National
% University
\begin{flushleft}
\hspace{0.3cm} \underline{Animal Breeding \& Genetics Group}
\end{flushleft}
%\vspace{0.5cm}
\begin{list2}
\item Animal Breeding \& Genetics Seminars organizer \\
  Department of Animal Science, University of Nebraska-Lincoln   \hfill \textbf{Spring 2016}\\

\item Animal Breeding \& Genetics Seminars organizer \\
  Department of Animal Science, University of Nebraska-Lincoln   \hfill \textbf{Fall 2015}\\
\end{list2}






\vspace{0.5cm}
\section{\sc OSS contributions} 
\begin{list1}
\item[] R packages
\begin{list2}
\item \textcolor{blue}{\href{http://cran.r-project.org/web/packages/dkDNA/index.html}{dkDNA}}: http://cran.r-project.org/web/packages/dkDNA/index.html
\end{list2}
\vspace{0.3cm}
\item[] Bioconductor packages
\begin{list2}
\item \textcolor{blue}{\href{http://bioconductor.org/packages/release/bioc/html/meshr.html}{meshr}}
\end{list2}
\vspace{0.3cm}
\begin{list2}
\item  \textcolor{blue}{\href{http://bioconductor.org/packages/release/data/annotation/html/MeSH.db.html}{MeSH.db}}
\end{list2}
\vspace{0.3cm}
\begin{list2}
\item \textcolor{blue}{\href{http://www.bioconductor.org/packages/release/data/annotation/html/MeSH.AOR.db.html}{MeSH.AOR.db}}
\end{list2}
\vspace{0.3cm}
\begin{list2}
\item \textcolor{blue}{\href{http://www.bioconductor.org/packages/release/data/annotation/html/MeSH.PCR.db.html}{MeSH.PCR.db}}
\end{list2}
\vspace{0.3cm}
\begin{list2}
\item \textcolor{blue}{\href{http://www.bioconductor.org/packages/release/data/annotation/}{MeSH.XXX.eg.db }} (84 packages)
\end{list2}
\begin{multicols}{2}
%\interlinepenalty=100000
\begin{itemize}
\item MeSH.Aca.eg.db
\item MeSH.Aga.PEST.eg.db
\item MeSH.Ame.eg.db
\item MeSH.Aml.eg.db
\item MeSH.Ana.eg.db
\item MeSH.Ani.FGSC.eg.db
\item MeSH.Ath.eg.db
%\item MeSH.Atu.K84.eg.db
\item MeSH.Bfl.eg.db
\item MeSH.Bsu.168.eg.db
%\item MeSH.Bsu.Bsn5.eg.db 
%\item MeSH.Bsu.RONN1.eg.db
\item MeSH.Bsu.TUB10.eg.db
%\item MeSH.Bsu.W23.eg.db  
\item MeSH.Bta.eg.db 
\item MeSH.Cal.SC5314.eg.db
\item MeSH.Cbr.eg.db 
\item MeSH.Cel.eg.db
\item MeSH.Cfa.eg.db
\item MeSH.Cin.eg.db
\item MeSH.Cja.eg.db
\item MeSH.Cpo.eg.db
\item MeSH.Cre.eg.db  
\item MeSH.Dan.eg.db
\item MeSH.Dda.3937.eg.db
\item MeSH.Ddi.AX4.eg.db
\item MeSH.Der.eg.db 
\item MeSH.Dgr.eg.db 
\item MeSH.Dme.eg.db 
\item MeSH.Dmo.eg.db 
\item MeSH.Dpe.eg.db
\item MeSH.Dre.eg.db
\item MeSH.Dse.eg.db
\item MeSH.Dsi.eg.db
\item MeSH.Dvi.eg.db  
\item MeSH.Dya.eg.db
%\item MeSH.Eco.536.eg.db    
\item MeSH.Eco.55989.eg.db
%\item MeSH.Eco.APEC01.eg.db 
%\item MeSH.Eco.B.REL606.eg.db
%\item MeSH.Eco.BW2952.eg.db 
\item MeSH.Eco.CFT073.eg.db
%\item MeSH.Eco.E24377A.eg.db   
\item MeSH.Eco.ED1a.eg.db 
\item MeSH.Eco.HS.eg.db
\item MeSH.Eco.IAI1.eg.db 
\item MeSH.Eco.IAI39.eg.db 
\item MeSH.Eco.K12.DH10B.eg.db
\item MeSH.Eco.K12.MG1655.eg.db
%\item MeSH.Eco.KO11FL.eg.db 
%\item MeSH.Eco.O103.H2.12009.eg.db
%\item MeSH.Eco.O111.H.11128.eg.db 
\item MeSH.Eco.O127.H6.E2348.69.eg.db
%\item MeSH.Eco.O157.H7.EC4115.eg.db
\item MeSH.Eco.O157.H7.EDL933.eg.db 
\item MeSH.Eco.O157.H7.Sakai.eg.db
%\item MeSH.Eco.O157.H7.TW14359.eg.db
%\item MeSH.Eco.O26.H11.11368.eg.db 
%\item MeSH.Eco.O26.H7.CB9615.eg.db 
\item MeSH.Eco.S88.eg.db
%\item MeSH.Eco.SE11.eg.db 
%\item MeSH.Eco.SMS35.eg.db 
\item MeSH.Eco.UMN026.eg.db
%\item MeSH.Eco.UTI89.eg.db 
\item MeSH.Eqc.eg.db 
\item MeSH.Gga.eg.db
\item MeSH.Gma.eg.db 
\item MeSH.Hsa.eg.db
\item MeSH.Laf.eg.db 
\item MeSH.Lma.eg.db 
\item MeSH.Mdo.eg.db 
\item MeSH.Mes.eg.db 
\item MeSH.Mga.eg.db 
\item MeSH.Miy.eg.db 
\item MeSH.Mml.eg.db 
\item MeSH.Mmu.eg.db 
\item MeSH.Mtr.eg.db 
\item MeSH.Nle.eg.db 
\item MeSH.Oan.eg.db 
\item MeSH.Ocu.eg.db 
\item MeSH.Oni.eg.db 
\item MeSH.Osa.eg.db 
\item MeSH.Pab.eg.db 
%\item MeSH.Pae.LESB58.eg.db 
%\item MeSH.Pae.PA14.eg.db
%\item MeSH.Pae.PA7.eg.db
\item MeSH.Pae.PAO1.eg.db 
\item MeSH.Pfa.3D7.eg.db
\item MeSH.Pto.eg.db 
\item MeSH.Ptr.eg.db 
\item MeSH.Rno.eg.db  
%\item MeSH.Sau.COL.eg.db 
%\item MeSH.Sau.ED98.eg.db
%\item MeSH.Sau.M013.eg.db 
%\item MeSH.Sau.MRSA252.eg.db 
%\item MeSH.Sau.MSHR1132.eg.db
%\item MeSH.Sau.MSSA476.eg.db
%\item MeSH.Sau.Mu3.eg.db
%\item MeSH.Sau.Mu50.eg.db 
%\item MeSH.Sau.MW2.eg.db  
%\item MeSH.Sau.N315.eg.db 
%\item MeSH.Sau.Newman.eg.db 
%\item MeSH.Sau.RF122.eg.db 
%\item MeSH.Sau.USA300FPR3757.eg.db 
\item MeSH.Sau.USA300TCH1516.eg.db 
%\item MeSH.Sau.VC40.eg.db
\item MeSH.Sce.S288c.eg.db 
\item MeSH.Sco.A32.eg.db 
\item MeSH.Sil.eg.db 
\item MeSH.Spo.972h.eg.db
\item MeSH.Spu.eg.db
\item MeSH.Ssc.eg.db 
\item MeSH.Syn.eg.db 
\item MeSH.Tbr.9274.eg.db 
\item MeSH.Tgo.ME49.eg.db
\item MeSH.Tgu.eg.db 
\item MeSH.Vvi.eg.db 
\item MeSH.Xla.eg.db  
\item MeSH.Xtr.eg.db 
\item MeSH.Zma.eg.db
\end{itemize}
\end{multicols}


\vspace{0.3cm}
\item[] Github
\begin{list2}
\item \textcolor{blue}{\href{https://github.com/morota}{https://github.com/morota}} 
\end{list2}
\end{list1}






\vspace{0.5cm}
\section{ \sc Additional training}
\vspace{1cm}
\section{\sc 2014-2015}
\begin{list2}
\item Participant of the Research Development Fellows Program (RDFP) 
\end{list2}

\section{\sc 2014}
\begin{list2}
\item Participant of Fall 2014 Adopting Research Based Instructional Strategies for Enhancing (ARISE) Professional Development Programs - Just in Time Teaching (JiTT)
\end{list2}




\vspace{0.5cm}
\section{\sc Participation in meetings, symposiums and workshops} 
\vspace{2cm}


\section{\sc 2015}
\begin{list2}

\item The 29th International Mammalian Genome Conference. Yokohama Port Opening Memorial Hall, Yokohama, Japan. November 8-11. 

  \vspace{0.5cm}
  
  \item DNA Technology: Where we've been, where we are, and where we're headed. The US Meat Animal Research Center, Clay Center, NE. October 19. 
  
  \vspace{0.5cm}
  
\item GO-FAANG Workshop. National Academy of Sciences Building, Washington, DC. October 7-8. 
\end{list2}  

\section{\sc 2014}
\begin{list2}

\item Sheep Genomics Workshop. University of Nebraska-Lincoln. November 13-14.  

\vspace{0.5cm}

\item NCERA-225 Meeting. Implementation and Strategies for National Beef Cattle Genetic Evaluation. Bozeman, MT. October 23-24. 

\vspace{0.5cm}

\item 19th Summer Institute in Statistical Genetics: 
``Module 23: Advanced Quantitative Genetics ". University of Washington.  July 23-25.
Taught by Mike Goddard and Peter Visscher. 

\vspace{0.5cm}

\item 19th Summer Institute in Statistical Genetics: 
``Module 19: Statistical \& Quantitative Genetics of Disease". University of Washington. July 21-23.
Taught by John Witte and Naomi Wray. 

\vspace{0.5cm}

\item UC Davis Bioinformatics Training Program: 
``Using Galaxy for Analysis of High Throughput Sequence Data". University of California, Davis. June 16-20. 
Taught by the Bioinformatics Core. 

\vspace{0.5cm}

\item Short course: ``Evolutionary Quantitative Genetics". University of Wisconsin-Madison. May 19-23. 
Taught by Bruce Walsh. 

\end{list2}  


\section{\sc 2013}
\begin{list2}
\item Short course: ``Statistical methods for prediction of complex traits using whole-genome molecular markers". University of Wisconsin-Madison. May 27-31. 
Taught by Daniel Gianola and Gustavo de los Campos.
\end{list2}  

\section{\sc 2012}
\begin{list2}
\item Short course: ``Introduction to genome-enabled selection \& Inferring causal phenotype networks using structural equation models". 
Kyoto University. August 31. Taught by Guilherme J.M. Rosa. 

\vspace{0.5cm}

\item Short course: ``Identifying Genes for Complex and Mendelian Traits Using Next Generation Sequence Data". 26th International Biometric Conference. August 26. Taught by Suzanne Leal.  

\vspace{0.5cm}

\item Short course: ``Programming and computer algorithms with focus on genomic selection in animal breeding".  University of Georgia. May 15 - June 1.  
Taught by Ignacy Misztal, Shogo Tsuruta, Ignacio Aguilar, Zulma Vitezica, and  Andres Legarra. 
\end{list2}  

\section{\sc 2009}
\begin{list2}
\item Symposium: ``Statistical Genetics of Livestock for the Post-Genomic Era (SGLPGE)". University of Wisconsin-Madison. May 4-6. 
\end{list2}  

\section{\sc 2006}
\begin{list2}
\item Short course: ``Estimation of Variance Conponents in Animal Breeding". Obihiro University of Agriculture and Veterinary Medicine. November. 
Taught by Shogo Tsuruta.  
\end{list2}  









\vspace{0.5cm}
\section{\sc Computer Skills} 
\begin{list2}
\item Programming Languages: Python, Fortran
\vspace{0.3cm}
\item Statistical/Numerical Computational  Tools: R, Octave
\vspace{0.3cm}
\item Content-description Languages: XML, XHTML, CSS, \LaTeX
\vspace{0.3cm}
\item Operating Systems: Linux and Mac OS X
\end{list2}

\vspace{0.5cm}
\section{\sc Miscellaneous} 
\begin{list2}
\item Languages: English and Japanese \\ 
\item Courses taken for credits at the University of Wisconsin-Madison 
\begin{itemize}

\item Spring 2012
\begin{itemize}
\item    Animal Sciences 875-004: Topics in Analysis of Quantitative Genomic Data (Daniel Gianola)
\item    Dairy Science 875-005: Parallel Programming \& High Performance Computing (Xiao-Lin Nick Wu)
\end{itemize}


\item Fall 2011
\begin{itemize}
\item    Dairy Science 875-005: Molecular Aspects of Animal Breeding (Hasan Khatib)
\item    Statistics 840: Statistical Model Building and Learning (Grace Wahba)
\end{itemize}



\item Spring 2011
\begin{itemize}
\item    Mathematics 609: Mathematical Methods in Systems Biology (Gheorghe Craciun)
\item    Statistics 610: Introduction to Statistical Inference (Chunming Zhang)
\item    Statistics 992-001: Statistical Methods for QTL Mapping (Karl Broman)
\end{itemize}


\item Fall 2010
\begin{itemize}
\item    Statistics 609: Mathematical Statistics I (Chunming Zhang)
\item    Statistics 701: Applied Time Series Analysis, Forecasting \& Control I (Yazhen Wang)
\item    Statistics 775: Introduction to Bayesian Decision \& Control (Kam-Wah Tsui)
\end{itemize}


\item Summer 2010
\begin{itemize}
\item    Population Health Sciences 904: Analytic Methods in Genetic Epidemiology (Corinne Engelman, Karl Broman, Bret Payseur, Kristin Meyers)
\end{itemize}


\item Spring 2010
\begin{itemize}

\item    Animal Sciences 875: Linear Models with Applications in Biology and Agriculture (Daniel Gianola)
\item    Statistics 850: Theory \& Application of Regression and Analysis of Variance II (Wei-Yin Loh)

\end{itemize}


\item Fall 2009
\begin{itemize}
\item    Computer Science 576: Introduction to Bioinformatics (Colin Dewey)
\item    Dairy Science 875-006: Design \& Analysis of Microarray Experiments in Agriculture (Guilherme J. M. Rosa)
\item    Dairy Science 875-011: Introduction to Bayesian Data Analysis with R (Xiao-Lin Nick Wu)
\item    Genetics 629: Evolutionary Genetics (John Doebley, Bret Larget, Bret Payseur)
\item    Statistics 849: Theory \& Application of Regression and Analysis of Variance I (Sunduz Keles)
\end{itemize}




\item Summer 2009
\begin{itemize}
\item Computer Science 367: Introduction to Data Structure
\end{itemize}

\item Spring 2009
\begin{itemize}
\item Agronomy 771: Experimental Design (Mike Casler)
\item Agronomy 772: Applications in ANOVA (Mike Casler)
\item Mathematics 222: Calculus and Analytic Geometry
\item Statistics 771: Statistical Computing (Michael Newton)
\end{itemize}


\item Fall 2008
\begin{itemize}
\item Statistics 424: Statistical Experimental Design for Engineers (Peter Z. G. Qian)
\item Statistics 541: Introduction to Biostatistics (Ismor Fischer)
\item Zoology 645: Modeling in Population Genetics \& Evolution (Andrew Peters)
\end{itemize}

\item Summer 2008
\begin{itemize}
\item Computer Science 302: Introduction to Programming
\item Mathematics 431: Introduction to the Theory of Probability
\end{itemize}

\end{itemize}
\end{list2}



\vspace{0.5cm}
\section{\sc References}
References and additional information available upon request. 

\begin{comment}
\begin{tabular}{ll}
% Sewall Wright Professor of Animal Breeding and Genetics
[1] & {\bf \underline{Daniel Gianola}}, Ph.D. Sewall Wright Professor of Animal Breeding and Genetics. \\
    & Department of Animal Sciences, University of Wisconsin-Madison. \\ 
    & Address: 1675 Observatory Dr. Madison, WI 53706-1284.  \\
    & E-mail: gianola -at- ansci.wisc.edu, Phone: 608-265-2054
\end{tabular}
\end{comment}



\end{resume}
\end{document}




