\documentclass[margin,line,10pt]{res}

\usepackage{verbatim}
\usepackage[hidelinks]{hyperref}
\usepackage{color}
\usepackage{multicol}
\usepackage{url}
\newcommand\doilink[1]{\href{http://dx.doi.org/#1}{#1}}
\newcommand\doi[1]{doi:\doilink{#1}}


\oddsidemargin -.5in
\evensidemargin -.5in
\textwidth=6.0in
\itemsep=0in
\parsep=0in


\newenvironment{list1}{
  \begin{list}{\ding{113}}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in} 
      \setlength{\leftmargin}{0.17in}}}{\end{list}}

\newenvironment{list2}{
  \begin{list}{$\bullet$}{%
      \setlength{\itemsep}{0in}
      \setlength{\parsep}{0in} \setlength{\parskip}{0in}
      \setlength{\topsep}{0in} \setlength{\partopsep}{0in} 
      \setlength{\leftmargin}{0.2in}}}{\end{list}}




\begin{document}

\name{Gota Morota \hspace{12.2cm} December 2019 \vspace*{.1in}}

\begin{resume}
\section{\sc Contact Information}
\vspace{.05in}
\begin{tabular}{@{}p{2in}p{4in}}
368 Litton Reaves Hall  & \hspace{2.5cm} {\it E-mail:}  \href{mailto:morota@vt.edu}{morota@vt.edu} \\       
 175 West Campus Drive  & \hspace{2.5cm} {\it Phone:} (540)-231-4732\\     
Virginia Tech  & \hspace{2.5cm} {\it WWW:} \textcolor{blue}{\href{http://morotalab.org/}{morotalab.org}  }\\
Blacksburg, Virginia 24061 USA  & \\
\end{tabular}


\vspace{0.4cm}
\section{\sc Research Interests}
I am a quantitative geneticist interested in incorporating statistics, machine learning, and bioinformatics to the study of animal and plant genetics in the omics era. The core line of my research is connecting the theory of statistical quantitative genetics to currently available molecular information. 
I am particularly interested in statistical methods for prediction of complex traits using whole-genome molecular markers. 
%In particular, I am working on integrating the different types of omics data for the prediction of complex traits using a broad range of agricultural species. 
% <li>Quantitative genetics and its application to animal breeding</li>


\vspace{0.4cm}
\section{\sc Education}

{\bf University of Wisconsin-Madison}, Madison, Wisconsin USA\\
\vspace*{-.1in}
\begin{list1}
\item[] Ph.D., Animal Sciences, May 2014
\begin{list2}
\vspace*{.05in}
\item Dissertation: ``Whole-Genome Prediction of Complex Traits Using Kernel Methods." 
\item Advisor: Prof. Dr. Daniel Gianola 
\item Committee: Drs. Corinne D. Engelman, Guilherme J. M. Rosa, Grace Wahba and Kent A. Weigel
\item Available at  \textcolor{blue}{\href{https://search.library.wisc.edu/catalog/9910205835702121}{UW-Madison Libraries}} 
\end{list2}
\vspace*{.05in}
\end{list1}





{\bf University of Wisconsin-Madison}, Madison, Wisconsin USA\\
\vspace*{-.1in}
\begin{list1}
\item[] M.S., Dairy Science, December 2011
\begin{list2}
\vspace*{.05in}
\item Thesis: ``Application of Bayesian and Sparse Network Models for Assessing Linkage Disequilibrium in Animals and Plants." 
\item Advisor: Prof. Dr. Daniel Gianola 
\item Committee: Drs. Guilherme J. M. Rosa and Kent A. Weigel
\end{list2}
\vspace*{.05in}
\end{list1}


{\bf Obihiro University of Agriculture and Veterinary Medicine}, Obihiro, Hokkaido Japan\\
\vspace*{-.1in}
\begin{list1}
\item[] B.S., Agricultural Science,  March 2008
\begin{list2}
\vspace*{.05in}
\item Thesis:  ``Genetic Analysis of Threshold Traits by Simulation and Real Data" 
\item Advisor: Prof. Dr. Mitsuyoshi Suzuki
\end{list2}
\end{list1}




\vspace{0.4cm}
\section{\sc Professional Positions}


Department of Animal and Poultry Sciences\\
{\bf Virginia Polytechnic Institute and State University}, Blacksburg, Virginia USA
\vspace{-.3cm}

Assistant Professor of Quantitative Genetics   \hfill {\bf 08/2018 - Present}\\
FTE: 70\% Research \& 30\% Teaching \\

\begin{itemize}
\item Genetics, Bioinformatics, and Computational Biology Program Faculty Member  \hfill {\bf 11/2019 - Present}\\
\item Translational Plant Sciences Program Faculty Member  \hfill {\bf 03/2019 - Present}\\
\end{itemize}

Department of Animal Science\\
{\bf University of Nebraska-Lincoln}, Lincoln, Nebraska USA
\vspace{-.4cm}

Assistant Professor of Theoretical Quantitative Genetics   \hfill {\bf 08/2014 - 07/2018}\\
FTE: 70\% Research \& 30\% Teaching \\



\vspace{0.4cm}
\section{\sc Visiting \& Temporary Positions}
Department of Animal Science\\
{\bf University of Nebraska-Lincoln}, Lincoln, Nebraska USA
\vspace{-.4cm}

Adjunct Professor  \hfill {\bf 09/2018 - 12/2019}\\ 



Laboratory of Biometry and Bioinformatics\\
Department of Agricultural and Environmental Biology \\
Graduate School of Agriculture and Life Science \\
{\bf The University of Tokyo}, Bunkyo, Tokyo, Japan
\vspace{-.3cm}

JST-CREST International Visiting Research Fellow  \hfill {\bf 09/2018 - 12/2018}\\
Host: Dr. Hiroyoshi Iwata
% Oversea


%Affiliate Faculty Member \\
%Department of xxx\\
%{\bf University of Nebraska-Lincoln}, Lincoln, Nebraska USA
  


\vspace{0.4cm}
\section{\sc Work \phantom{1cm} Experience}
Department of Animal Sciences\\
{\bf University of Wisconsin-Madison}, Madison, Wisconsin USA


\vspace{-.4cm}

Graduate Research Assistant   \hfill {\bf 06/2011 - 05/2014}\\
\vspace{-.4cm}

Animal Genetics Research \& Development  Group \\
{\bf Zoetis, Inc.}, Kalamazoo, Michigan USA

\vspace{-.4cm}

Quantitative Geneticist (student internship)   \hfill {\bf 06/2013 - 11/2013}\\








\vspace{0.5cm}
\section{\sc Professional society memberships}
\begin{list2}
  % North American Plant Phenotyping Network. 2018 - Present
  % International Society of Precision Agriculture
  % American Society of Agricultural and Biological Engineers
  % Society of Applied Spectroscopy
  \item Crop Science Society of America. 2019 - Present
    \vspace{0.3cm}
  \item American Statistical Association. 2018 - Present
    \vspace{0.3cm}
  \item  The Japanese Society of Breeding. 2018 - Present
  \vspace{0.3cm}
\item  The Genetics Society of America. 2016 - Present
  \vspace{0.3cm}
\item  The Japanese Society of Animal Science. 2016 - Present
  \vspace{0.3cm}
  %\item  The International Society for Animal Genetics. 2015 - Present
  %  \vspace{0.3cm}
\item The American Society of Animal Science. 2014 - Present
  \vspace{0.3cm}
\item The International Biometric Society (ENAR). 2012 - Present
\end{list2}



\vspace{0.5cm}
\section{\sc Editorial Activities}
%\vspace{.5cm}
%My Publons account \textcolor{blue}{\href{https://publons.com/author/1316321}{https://publons.com/author/1316321}. }
%\vspace{0.5cm}
%\\

\underline{Associate Editor}
\begin{itemize}
\item BMC Genetics, September 2019 - Present
  \begin{itemize}
    \item Number of manuscripts handled: 2019 (1)
  \end{itemize}
\end{itemize}
\vspace{0.3cm}

\underline{Guest Associate Editor}
\begin{itemize}
\item Frontiers in Genetics - Livestock Genomics, July 2019 - December 2020 \\
  Research Topic: High-throughput phenotyping in the genomic improvement of livestock
  \begin{itemize}
    \item Number of manuscripts handled: 2019 (1)
  \end{itemize}
\end{itemize}
\vspace{0.3cm}

\underline{Editorial Board}
 \vspace{.3cm}
\begin{itemize}
\item Journal of Animal Science, July 2017 - July 2020 
\end{itemize}
  
\vspace{0.3cm}
\underline{Reviewed for}
\begin{itemize}
    \vspace{.1cm}
\item Animal (1), Animal Genetics (3), Animal Production Science (2), Animal Science Journal (1), Bioinformatics (3), BMC Bioinformatics (1), BMC Genetics (6), BMC Genomics (3), BMC Plant Biology (1), Computers and Electronics in Agriculture (1), Crop Science (1), DNA Research (1), Functional \& Integrative Genomics (1), Frontiers in Genetics (3), G3: Genes, Genomes, Genetics (1), Genetics (2), Genetics Selection Evolution (4), Heredity (1), Journal of Agricultural, Biological, and Environmental Statistics (1), Journal of Animal Breeding and Genetics (7), Journal of Animal Science (19), Journal of Animal Science and Biotechnology (3), Journal of Dairy Science (5), Livestock Science (5), Meat Science (1), New Phytologist (1), PLOS ONE (4), Poultry Science (2), Scientia Agricola (3), Scientific Reports (1), Theoretical and Applied Genetics (4), Theoretical Population Biology (1), The Plant Genome (1) 
  \vspace{.1cm}
  \item Number of manuscripts reviewed: 2019 (27), 2018 (19), 2017 (20), 2016 (10), 2015 (10), 2014 (6), 2013 (1), 2012 (1)
\end{itemize}
% Total: 27


\begin{comment}
\vspace{0.5cm}
\section{\sc Manuscripts under review}

\begin{list1}

  \item [{\bf 30}.] Dhatt B, Paul P, Sandhu J, Hussain W, Irvin L, Zhu F, Adviento-Borbe M, Lorence A, Staswick P, Yu H, \textbf{\underline{Morota G}}, and Walia H.  Allelic Variation in Rice Fertilization Independent Endosperm 1 Contributes to Grain Width Under High Night Temperature Stress \emph{New Phytologist}. 

    
\item [{\bf 30}.] Pegolo S, Momen M, \textbf{\underline{Morota G}}, Gianola G, Rosa GJM, Bittante G, and Cecchinato A. Structural equation modeling for investigating multi-trait genetic architecture of udder health in dairy cattle. \emph{Scientific Reports}. 

  \item  [{\bf 29}.] Roudbar MA, Mohammadabadi MR, Mehrgardi AA, Abdollahi-Arpanahi R,  Momen M, \textbf{\underline{Morota G}}, Lopes FB, Gianola D, and Rosa GJM. Integration of single nucleotide variants and whole-genome DNA methylation profiles for classification of rheumatoid arthritis cases from controls. \emph{Scientific Reports}

  \item  [{\bf 29}.]  Rovadoscki GA, Pertille SFN, Moreira GCM, P{\'e}rtille F,  Botelho AA, Cesar ASM, Petrini J, Gerv{\'a}sio IC, Dauria BD, \textbf{\underline{Morota G}}, Spangler ML, Pinto LFB, de Carvalho GGP, Lanna DPD, Coutinho LL, and Mour{\~a}o GB. Estimates of genomic heritability and genome-wide association study for meat quality traits in Santa In{\^e}s sheep. \emph{animal}

  \item  [{\bf 28}.]  Mamani GCM, Santana BF, Oliveira Junior GA,  Mattos EC, Eler JP, Ventura RV, \textbf{\underline{Morota G}}, and Ferraz JBS  \emph{PLoS ONE}.     

    
    \end{list1}
\end{comment}


%\begin{comment}
\vspace{0.5cm}
\section{\sc Preprints}
\begin{list1}
  
\item [{\bf 41}.] Campbell MT, Yu H, Momen M, and \textbf{\underline{Morota G}}. Examining the relationships between phenotypic plasticity and local environments with genomic structural equation models. \emph{bioRxiv}. doi: \textcolor{blue}{\href{https://doi.org/10.1101/2019.12.11.873257}{10.1101/2019.12.11.873257}}


    \vspace{0.5cm}

\item [{\bf 40}.] Wang Z, Chapman D, \textbf{\underline{Morota G}}, and Cheng H. A Multiple-trait Bayesian variable selection regression method for integrating phenotypic causal networks in genome-wide association studies. \emph{bioRxiv}. doi: \textcolor{blue}{\href{https://doi.org/10.1101/847285}{10.1101/847285}}

  \vspace{0.5cm}
    
\item  [{\bf 39}.] Yu H and \textbf{\underline{Morota G}}. GCA: An R package for genetic connectedness analysis using pedigree and genomic data. \emph{bioRxiv}.  doi: \textcolor{blue}{\href{https://doi.org/10.1101/696419}{10.1101/696419}}

  \vspace{0.5cm}

\item  [{\bf 38}.]  Campbell MT, Grondin A, Walia H, and \textbf{\underline{Morota G}}. Leveraging genome-enabled growth models to study shoot growth responses to water deficit in rice ({\it Oryza sativa}). \emph{bioRxiv}.  doi: \textcolor{blue}{\href{https://doi.org/10.1101/690479}{10.1101/690479}}

  \vspace{0.5cm}
     
\item  [{\bf 37}.]  Hussain W, Campbell MT, Walia H, and \textbf{\underline{Morota G}}. Variance heterogeneity genome-wide mapping for cadmium in bread wheat reveals novel genomic loci and epistatic interactions. \emph{bioRxiv}.  doi: \textcolor{blue}{\href{https://doi.org/10.1101/668087}{10.1101/668087}}

  \vspace{0.5cm}
  
\item  [{\bf 36}.] \textbf{\underline{Morota G}}, Jarquin D, Campbell MT, and Iwata H. Statistical methods for the quantitative genetic analysis of high-throughput phenotyping data. \emph{arXiv}.  \textcolor{blue}{\href{https://arxiv.org/abs/1904.12341}{https://arxiv.org/abs/1904.12341}}

 
\end{list1}
%\end{comment}

  
\vspace{0.5cm}


\section{\sc Peer reviewed journal papers}
\vspace{0.5cm}
10 first author, 15 co-author, and 9 senior author publications (22 publications as a PI)\\
% 12 publications prior to becoming a PI

\section{\sc 2020}
\begin{list1}

  \item  [{\bf 35}.] Baba T, Momen M, Campbell, MT, Walia H, and \textbf{\underline{Morota G}}. Multi-trait random regression models increase genomic prediction accuracy for a temporal physiological trait derived from high-throughput phenotyping. \emph{PLOS ONE}. In press. doi: \textcolor{blue}{\href{https://doi.org/10.1101/772038}{10.1101/772038}}
  
  \vspace{0.5cm}

\item [{\bf 34}.] Paul P, Dhatt B, Sandhu J, Hussain W, Irvin L, \textbf{\underline{Morota G}}, Staswick P, and Walia H. Divergent phenotypic response of rice accessions to transient heat stress during early seed development. \emph{Plant Direct}. In press.

  \end{list1}


\section{\sc 2019}
\begin{list1}
  
\item  [{\bf 33}.] Momen M, Campbell MT, Walia H, and \textbf{\underline{Morota G}}. Utilizing trait networks and structural equation models as tools to interpret multi-trait genome-wide association studies. \emph{Plant Methods}. \textbf{15}:107. doi: \textcolor{blue}{\href{https://doi.org/10.1186/s13007-019-0493-x}{10.1186/s13007-019-0493-x}}

     \vspace{0.5cm}
     
\item  [{\bf 32}.]  Momen M, Campbell MT, Walia H, and \textbf{\underline{Morota G}}. 2019. Predicting longitudinal traits derived from high-throughput phenomics in contrasting environments using genomic Legendre polynomials and B-splines. \emph{G3: Genes, Genomes, Genetics}. 9:3369-3380.  doi: \textcolor{blue}{\href{https://doi.org/10.1534/g3.119.400346}{10.1534/g3.119.400346}}

     \vspace{0.5cm}

\item  [{\bf 31}.] Yu H, Campbell MT, Zhang Q, Walia H, and \textbf{\underline{Morota G}}. 2019. Genomic Bayesian confirmatory factor analysis and Bayesian network to characterize a wide spectrum of rice phenotypes. \emph{G3: Genes, Genomes, Genetics}. \textbf{9}:1975-1986. doi: \textcolor{blue}{\href{https://doi.org/10.1534/g3.119.400154}{10.1534/g3.119.400154}}

  \vspace{0.5cm}
  
\item  [{\bf 30}.] Campbell MT, Momen M, Walia H, and \textbf{\underline{Morota G}}. 2019. Leveraging breeding values obtained from random regression models for genetic inference of longitudinal traits. \emph{The Plant Genome}. \textbf{12}:180075. doi: \textcolor{blue}{\href{https://doi.org/10.3835/plantgenome2018.10.0075}{10.3835/plantgenome2018.10.0075}}

\end{list1}


\section{\sc 2018}
\begin{list1}

\item  [{\bf 29}.]  Hussain W, Campbell MT, Walia H, and \textbf{\underline{Morota G}}. 2018. ShinyAIM: Shiny-based application of interactive Manhattan plots for longitudinal genome-wide association studies. \emph{Plant Direct}. \textbf{2}:1-4. doi: \textcolor{blue}{\href{https://doi.org/10.1002/pld3.91}{10.1002/pld3.91}}

   \vspace{0.5cm}
  
\item  [{\bf 28}.] Momen M, Mehrgardi AA, Roudbar MA, Kranis A, Pinto RM, Valente BD, \textbf{\underline{Morota G}}, Rosa GJM, and Gianola D. 2018. Including phenotypic causal networks in genome-wide association studies using mixed effects structural equation models. \emph{Frontiers in Genetics}. \textbf{9}:455. doi: \textcolor{blue}{\href{https://doi.org/10.3389/fgene.2018.00455}{10.3389/fgene.2018.00455}}

  \vspace{0.5cm}
  
\item  [{\bf 27}.] Momen M and  \textbf{\underline{Morota G}}. 2018. Quantifying genomic connectedness and prediction accuracy from additive and non-additive gene actions. \emph{Genetics Selection Evolution}. \textbf{50}:45. doi: \textcolor{blue}{\href{https://doi.org/10.1186/s12711-018-0415-9}{10.1186/s12711-018-0415-9}}

  \vspace{0.5cm}
  
\item  [{\bf 26}.] Yu H, Spangler ML, Lewis RM, and {\bf \underline{Morota G}}. 2018.  Do stronger measures of genomic connectedness enhance prediction accuracies across management units? \emph{Journal of Animal Science}. \textbf{96}:4490-4500.  doi: \textcolor{blue}{\href{https://doi.org/10.1093/jas/sky316}{10.1093/jas/sky316}} 

  \vspace{0.5cm}

\item  [{\bf 25}.] Momen M, Mehrgardi AA, Sheikhy ASA, Kranis A, Tusell L, \textbf{\underline{Morota G}}, Rosa GJM, and Gianola D. 2018. Predictive ability of genome-assisted statistical models under various forms of gene action. \emph{Scientific Reports}. \textbf{8}:12309. doi: \textcolor{blue}{\href{https://doi.org/10.1038/s41598-018-30089-2}{10.1038/s41598-018-30089-2}} 

  \vspace{0.5cm}

\item [{\bf 24}.] Campbell MT, Walia H, and \textbf{\underline{Morota G}}. 2018. Utilizing random regression models for genomic prediction of a longitudinal trait derived from high-throughput phenotyping. \emph{Plant Direct}. \textbf{2}:1-11. doi: \textcolor{blue}{\href{https://doi.org/10.1002/pld3.80}{10.1002/pld3.80}} 
  
  \vspace{0.5cm}

\item  [{\bf 23}.] Alvarenga AB, Rovadoscki GA, Petrini J, Coutinho LL, \textbf{\underline{Morota G}}, Spangler ML, Pinto LFB, Carvalho GGP, and Mour{\~a}o GB. 2018. Linkage disequilibrium in Brazilian Santa  In{\^e}s breed, Ovis aries. \emph{Scientific Reports}.  \textbf{8}:8851. doi: \textcolor{blue}{\href{https://doi.org/10.1038/s41598-018-27259-7}{10.1038/s41598-018-27259-7}} 

  \vspace{0.5cm}
  
\item  [{\bf 22}.] Rovadoscki GA, Pertille SFN, Alvarenga AB,  Cesar ASM, P{\'e}rtille F, Petrini J, Franzo V, Soares WVB, \textbf{\underline{Morota G}}, Spangler ML, Pinto LFB, de Carvalho GGP, Lanna DPD, Coutinho LL, and Mour{\~a}o GB. 2018. Estimates of genomic heritability and genome-wide association study for fatty acids profile in Santa In{\^e}s sheep. \emph{BMC Genomics}. \textbf{19}:375. doi: \textcolor{blue}{\href{https://doi.org/10.1186/s12864-018-4777-8}{10.1186/s12864-018-4777-8}} 
    
    \vspace{0.5cm}
    
\item  [{\bf 21}.] \textbf{\underline{Morota G}}, Ventura RV, Silva FF, Koyama M, and Fernando SC. 2018. Machine learning and data mining advance predictive big data analysis in precision animal agriculture. \emph{Journal of Animal Science}. \textbf{96}:1540-1550. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1093/jas/sky014}{10.1093/jas/sky014}}

  \vspace{0.5cm}
  
\item  [{\bf 20}.] He J, Xu J, Wu XL, Bauck S, Lee J, \textbf{\underline{Morota G}}, Kachman SD, and Spangler ML. 2018. Comparing strategies for selection of low-density SNPs for imputation-mediated genomic prediction in U.S. Holsteins. \emph{Genetica}. \textbf{146}:137-149. doi: \textcolor{blue}{\href{https://dx.doi.org/10.1007/s10709-017-0004-9}{10.1007/s10709-017-0004-9}}

\end{list1}


\section{\sc 2017}

\begin{list1}

\item  [{\bf 19}.] \textbf{\underline{Morota G}}. 2017. ShinyGPAS: Interactive genomic prediction accuracy simulator based on deterministic formulas. \emph{Genetics Selection Evolution}. \textbf{49}:91. doi: \textcolor{blue}{\href{https://dx.doi.org/10.1186/s12711-017-0368-4}{10.1186/s12711-017-0368-4}}.

  \vspace{0.5cm}
    
\item  [{\bf 18}.] Abdollahi-Arpanahi R, \textbf{\underline{Morota G}}, and  Pe\~{n}agaricano F. 2017. Predicting bull fertility using genomic data and biological information. \emph{Journal of Dairy Science}. \textbf{100}:9656-9666. doi: \textcolor{blue}{\href{https://doi.org/10.3168/jds.2017-13288}{10.3168/jds.2017-13288}}.

    \vspace{0.5cm}
    
  \item  [{\bf 17}.]  Yu H, Spangler ML, Lewis RM, and \textbf{\underline{Morota G}}. 2017. Genomic relatedness strengthens genetic connectedness across management units. \emph{G3: Genes, Genomes, Genetics}. \textbf{10}:3543-3556. doi: \textcolor{blue}{\href{https://doi.org/10.1534/g3.117.300151}{10.1534/g3.117.300151}}.

    \vspace{0.5cm}
    
\item [{\bf 16}.] Beissinger TM and \textbf{\underline{Morota G}}. 2017. Medical subject heading (MeSH) annotations illuminate maize genetics and evolution. \emph{Plant Methods}. \textbf{13}:8. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1186/s13007-017-0159-5}{10.1186/s13007-017-0159-5}}.

\end{list1}



\section{\sc 2016}

\begin{list1}

  \item [{\bf 15}.]  \textbf{\underline{Morota G}}, Beissinger TM, and Pe\~{n}agaricano F. 2016. MeSH-informed enrichment analysis and MeSH-guided semantic similarity among functional terms and gene products in chicken. \emph{G3: Genes, Genomes, Genetics}. \textbf{6}:2447-2453. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1534/g3.116.031096}{10.1534/g3.116.031096}}.  

\vspace{0.5cm}

  \item [{\bf 14}.] Abdollahi-Arpanahi R, {\bf \underline{Morota G}}, Valente BD, Kranis A, Rosa GJM, and Gianola D. 
  2016. Differential contribution of genomic regions to marked genetic variation and prediction of quantitative traits in broiler chickens. \emph{Genetics Selection Evolution}. \textbf{48}:10. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1186/s12711-016-0187-z}{10.1186/s12711-016-0187-z}}.  

\end{list1}
  
\section{\sc 2015}

\begin{list1}

\item [{\bf 13}.]  Hu Y, {\bf \underline{Morota G}}, Rosa GJM, and Gianola D. 2015. Prediction of plant height in Arabidopsis thaliana from DNA methylation data. \emph{Genetics}. \textbf{201}:779-793. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1534/genetics.115.177204}{10.1534/genetics.115.177204}}.  

\vspace{0.5cm}

\item [{\bf 12}.]  Valente BD, {\bf \underline{Morota G}}, Pe\~{n}agaricano F, Gianola D, Weigel KA, and Rosa GJM. 2015. The causal meaning of genomic predictors and how it affects  construction and comparison of genome-enabled selection models. \emph{Genetics}. \textbf{200}:483-494. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1534/genetics.114.169490}{10.1534/genetics.114.169490}}.  

\vspace{0.5cm}

\item [{\bf 11}.] {\bf \underline{Morota G}}, Pe\~{n}agaricano F, Petersen JL, Ciobanu DC, Tsuyuzaki K, and Nikaido I. 2015. An application of MeSH enrichment analysis in livestock. \emph{Animal Genetics}. \textbf{46}:381-387. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1111/age.12307}{10.1111/age.12307}}.  

\vspace{0.5cm}

\item [{\bf 10}.] Abdollahi-Arpanahi R, {\bf \underline{Morota G}}, Valente BD, Kranis A, Rosa GJM, and Gianola D. 2015. Assessment of bagging GBLUP for whole-genome prediction of broiler chicken traits. \emph{Journal of Animal Breeding and Genetics}. \textbf{132}:218-228. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1111/jbg.12131}{10.1111/jbg.12131}}.  

\vspace{0.5cm}

\item [{\bf 9}.]  Tsuyuzaki K, {\bf \underline{Morota G}}, Ishii M, Nakazato T, Miyazaki S, and Nikaido I. 2015. 
  MeSH ORA framework: R/Bioconductor packages to support MeSH over-representation analysis. \emph{BMC Bioinformatics}. {\bf 16}:45. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1186/s12859-015-0453-z}{10.1186/s12859-015-0453-z}}. 

\end{list1}


\section{\sc 2014}

\begin{list1}


\item [{\bf 8}.]  {\bf \underline{Morota G}} and Gianola D. 2014. 
  Kernel-based whole-genome prediction of complex traits: a review. \emph{Frontiers in Genetics}. {\bf 5}:363. doi: \textcolor{blue}{\href{http://dx.doi.org/10.3389/fgene.2014.00363}{10.3389/fgene.2014.00363}}. 

\vspace{0.5cm}

\item [{\bf 7}.]  {\bf \underline{Morota G}}, Boddhireddy P, Vukasinovic N, Gianola D, and DeNise S.  2014. Kernel-based variance components estimation and  whole-genome prediction of pre-corrected phenotypes and progeny tests for dairy cow health traits. \emph{Frontiers in Genetics}. {\bf 5}:56. doi: \textcolor{blue}{\href{http://dx.doi.org/10.3389/fgene.2014.00056}{10.3389/fgene.2014.00056}}. 

\vspace{0.5cm}

\item [{\bf 6}.]  {\bf \underline{Morota G}}, Abdollahi-Arpanahi R, Kranis A, and Gianola D. 2014.   
     Genome-enabled prediction of broiler traits in chickens using genomic annotation. \emph{BMC Genomics}. {\bf 15}:109. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1186/1471-2164-15-109}{10.1186/1471-2164-15-109}}. 
    

\vspace{0.5cm}

\item [{\bf 5}.]  Abdollahi-Arpanahi R, Pakdel A, Nejati-Javaremi A, Moradi-Shahrbabak M, 
     {\bf \underline{Morota G}}, Valente BD, Kranis A, Rosa GJM, and Gianola D. 2014.
     Dissection of additive genetic variability for quantitative traits in chickens using SNP markers. \emph{Journal of Animal Breeding and Genetics}. \textbf{131}:183-193. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1111/jbg.12079}{10.1111/jbg.12079}}. 

\vspace{0.5cm}

\item [{\bf 4}.]  Abdollahi-Arpanahi R,  Nejati-Javaremi A,  Pakdel A, Moradi-Shahrbabak M, 
     {\bf \underline{Morota G}}, Valente BD, Kranis A, Rosa GJM, and Gianola D. 2014.   
     Effect of allele frequencies, effect sizes and number of markers on prediction of quantitative 
     traits in chickens. \emph{Journal of Animal Breeding and Genetics}. \textbf{131}:123-133. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1111/jbg.12075}{10.1111/jbg.12075}}. 
\end{list1}



\section{\sc 2013}


\begin{list1}
\item [{\bf 3}.]  {\bf \underline{Morota G}}, Koyama M, Rosa GJM, Weigel KA, and Gianola D. 2013.
     Predicting complex traits using a diffusion kernel on genetic markers with an application to dairy cattle and wheat data. \emph{Genetics Selection Evolution}. {\bf 45}:17. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1186/1297-9686-45-17}{10.1186/1297-9686-45-17}}.  

\vspace{0.5cm}

\item [{\bf 2}.] {\bf \underline{Morota G}} and Gianola D. 2013.  Evaluation of linkage disequilibrium in wheat with an L1 regularized sparse Markov network.
  \emph{Theoretical and Applied Genetics}. {\bf 126}:1991-2002. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1007/s00122-013-2112-y}{10.1007/s00122-013-2112-y}}. 
\end{list1}


\section{\sc 2012}
\begin{list1} 
\item [{\bf 1}.] {\bf \underline{Morota G}}, Valente BD, Rosa GJM, Weigel KA, and Gianola D. 2012.  
An assessment of linkage disequilibrium in Holstein cattle using a Bayesian network. \emph{Journal of Animal Breeding and Genetics}. {\bf 129}:474-487. doi: \textcolor{blue}{\href{http://dx.doi.org/10.1111/jbg.12002}{10.1111/jbg.12002}}. 
\end{list1}



\vspace{1.0cm}
\section{\sc Papers in Proceedings}
\vspace{0.9cm}
6 co-author and 1 senior author

\section{\sc 2019}
\begin{list1}

\item [{\bf 7}.] Atagi Y, {\bf \underline{Morota G}}, Onogi A, Osawa T, Yasumori T, Adachi K, Yamaguchi S, Aihara M, Goto H, Togashi K, and Iwata H. 2019. Consideration of heat stress in multiple lactation test–day models for dairy production traits. \emph{Interbull Bulletin}. \textbf{55}:81-87. \textcolor{blue}{\href{https://journal.interbull.org/index.php/ib/article/view/1467}{HTML}}. 

\end{list1}


\section{\sc 2018}
\begin{list1}
  
\item [\bf{6}.] Yu H, Spangler ML, Lewis RM, and {\bf \underline{Morota G}}. 2018. 
Stronger measures of genomic connectedness enhance prediction accuracies across management units. In: \emph{Proceedings, 11th World Congress of Genetics Applied to Livestock Production}. \textbf{11}:406. February 11-16, Auckland, New Zealand. 
\textcolor{blue}{\href{http://www.wcgalp.org/proceedings/2018/stronger-measures-genomic-connectedness-enhance-prediction-accuracies-across}{PDF}}.  

\vspace{0.5cm}

\item [\bf{5}.] Abdollahi-Arpanahi R, {\bf \underline{Morota G}}, and Penagaricano F. Predicting bull fertility using biologically informed genomic models.  In: \emph{Proceedings, 11th World Congress of Genetics Applied to Livestock Production}. \textbf{11}:683. February 11-16. Auckland, New Zealand. \textcolor{blue}{\href{http://www.wcgalp.org/proceedings/2018/predicting-bull-fertility-using-biologically-informed-genomic-models}{PDF}}.  

  \vspace{0.5cm}
  
\item [\bf{4}.] Mamani GC, Santana BF, Oliveira Junior GA, Mattos E, Ventura RV, Eler JP, {\bf \underline{Morota G}}, and Ferraz JBS. In: \emph{Proceedings, 11th World Congress of Genetics Applied to Livestock Production}. \textbf{11}:855. February 11-16. Auckland, New Zealand. \textcolor{blue}{\href{http://www.wcgalp.org/proceedings/2018/effect-inbreeding-productive-traits-nellore-cattle}{PDF}}.  

  
\end{list1}

\section{\sc 2014}
\begin{list1}
\item [\bf{3}.] Gianola D, {\bf \underline{Morota G}}, and Crossa J. 2014. 
Genome-enabled Prediction of Complex Traits with Kernel Methods: What Have We Learned? In: \emph{Proceedings, 10th World Congress of Genetics Applied to Livestock Production}. August 17-22, Vancouver, BC, Canada. 
\textcolor{blue}{\href{http://www.morotalab.org/publications/pdf/gianola2014WCGALP.pdf}{PDF}}.  


\vspace{0.5cm}

\item [\bf{2}.] Valente BD, {\bf \underline{Morota G}}, Rosa GJM, Gianola D, and  Weigel KA. 2014. 
Causal meaning of genomic predictors: Implication on genome-enabled selection modeling. In: \emph{Proceedings, 10th World Congress of Genetics Applied to Livestock Production}. August 17-22, Vancouver, BC, Canada. 
\textcolor{blue}{\href{http://www.morotalab.org/publications/pdf/valente2014WCGALP.pdf}{PDF}}.  
\end{list1}


\section{\sc 2011}
\begin{list1}
\item [\bf{1}.] Bueno Filho JS*, {\bf \underline{Morota G}}*, Tran Q, Maenner MJ, Vera-Cala LM, Engelman CD, and Meyers KJ. 2011. Analysis of human mini-exome sequencing data from Genetic \& Analysis Workshop 17 using a  Bayesian hierarchical mixture model. In: \emph{BMC Proceedings}. {\bf 5}(Suppl 9):S93. DOI: \textcolor{blue}{\href{http://dx.doi.org/10.1186/1753-6561-5-S9-S93}{10.1186/1753-6561-5-S9-S93}}. *equal contribution.   
\end{list1}






%\section{\sc Articles and Editorials not peer reviewed}


\vspace{0.5cm}
\section{\sc Invited Presentations}
\vspace{0.5cm}
14 domestic and 13 international \\
\noindent
%My PDF presentation slides are available on \textcolor{blue}{\href{http://www.slideshare.net/chikudaisei/}{SlideShare}} (http://www.slideshare.net/chikudaisei/). \\

\section{\sc 2020}
\begin{list2}

\item [{\bf 27}.] TBA. Invited Session: Interactive visualization for effective decision-making in agricultural sciences. The 30th International Biometric Conference (2020IBC). Seoul, South Korea. July 5-10.

  \vspace{0.5cm}
    
\item [{\bf 26}.] Variance heterogeneity genome-wide mapping for cadmium in bread wheat reveals novel genomic loci and epistatic interactions. Plant Molecular Breeding Workshop. The Plant and Animal Genome XXVIII Conference. Town and Country Hotel, San Diego, CA. January 11-15.
  
\end{list2}



\section{\sc 2019}
\begin{list2}

\item [{\bf 25}.] Do structural equation models advance multi-trait genome-wide association analysis? Special Seminar. Bioscience and Biotechnology Center, Nagoya University, Nagoya, Japan. October 25. 
  
  \vspace{0.5cm}
  
   \item [{\bf 24}.] Variance heterogeneity association analysis in wheat to reveal novel genomic loci and epistatic interactions. Symposium on Statistical and Data Scientific Methods for Omics-data Analysis in Agricultural and Life Sciences. TKP Ochanomizu Conference Center, Tokyo, Japan. October 15.

     \vspace{0.5cm}
    
  \item [{\bf 23}.] Statistical methods for quantitative genetic analysis of high-throughput phenotyping data.  University of Florida Genetics Institute Seminar. University of Florida, Gainesville, FL. October 10.

    \vspace{0.5cm}
    
\item [{\bf 22}.]  Big data statistical techniques applied to precision animal nutrition and production. The 6th EAAP International Symposium on Energy and Protein Metabolism and Nutrition. Ouro Minas Palace Hotel, Belo Horizonte, MT, Brazil. September 9-12.

  \vspace{0.5cm}
  
 \item [{\bf 21}.] Statistical quantitative genetic modeling for image-based high-throughput phenotyping data. The 64th RBras (The Brazilian  Region  of the International Biometric Society) and 18th SEAGRO (Symposium on Statistics Applied to Agricultural Experimentation) Meeting. Centro de Eventos do Pantanal, Cuiab{\'a}, MT, Brazil. July 29 - August 2.

   \vspace{0.5cm}

   
 \item [{\bf 20}.]      
     Statistical methods for quantitative genetic analysis of high-throughput phenotyping data. Special seminar. Department of Statistics. Federal University of Vi\c cosa, Vi\c cosa, MG, Brazil. July 25.
   
    
   \vspace{0.5cm}
   
   \item [{\bf 19}.]  Multi-omic data integration in quantitative genetics. Breeding and Genetics Symposium: FAANG. ASAS-ADSA Midwest Joint Annual Meeting. CHI Health Center Convention, Omaha, NE. March 11-13.

  \vspace{0.5cm}

 \item [{\bf 18}.] Recent advances in Medical Subject Headings (MeSH) analysis. Cattle/Sheep/Goat 2 Workshop. The Plant and Animal Genome XXVII Conference. Town and Country Hotel, San Diego, CA. January 12-16.

\end{list2}

  
  
\section{\sc 2018}
\begin{list2}

  \item [{\bf 17}.] The role of interactive visualization in big data analysis and its application to plant breeding. The 8th Agrigenomic Industry Workshop. Co-working space Kayabacho Co-Edo, Chuo-ku, Tokyo, Japan. December 21.

    \vspace{0.5cm}
    
   \item [{\bf 16}.] Quantifying genomic connectedness and whole-genome prediction accuracy using bootstrap aggregation sampling. The 11th International Conference of the ERCIM WG on Computational and Methodological Statistics (CMStatistics 2018). University of Pisa, Pisa, Italy. December 14-16.

     \vspace{0.5cm}
     
    \item [{\bf 15}.] How big data, machine learning and bioinformatics are impacting genetic selection. Poultry Tech Summit. Georgia Tech Hotel \& Conference Center, Atlanta, GA. November 5-7.

      \vspace{0.5cm}
      
  \item [{\bf 14}.] Statistical learning in animal and plant breeding using multi-omic data. IX International Symposium on Genetics and Breeding (IX SIGM) / DuPont Plant Sciences Symposium.  Federal University of Vi\c cosa, Vi\c cosa, MG, Brazil. October 24-25.
  
  
  \vspace{0.5cm}
  
\item [{\bf 13}.] Bayesian genomic factor analysis and Bayesian network to characterize high-throughput phenotyping data. T-PIRC Symposium: Innovation for global food production towards sustainable future. The 2018 Tsukuba Global Science Week. Tsukuba International Congress Center, Tsukuba, Ibaraki, Japan. September 20-22.
  

  \vspace{0.5cm}
  
  
\item [{\bf 12}.] Do structural equation models advance genome-wide association analysis? Special seminar. School of Veterinary Medicine and Animal Science (FMVZ), University of S\~{a}o Paulo, Pirassununga, S\~{a}o Paulo, Brazil.  May 28.

    \vspace{0.5cm}
    
\item [{\bf 11}.]  Statistical and computational quantitative genetic analyses for genetic improvement of agricultural species. Special seminar.  Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA. February 23.

  \vspace{0.5cm}
  
\item [{\bf 10}.] Do stronger measures of genomic connectedness enhance prediction accuracies across management units?  Genomic Selection and Genome-Wide Association Studies Workshop. The Plant and Animal Genome XXVI Conference. Town and Country Hotel, San Diego, CA. January 13-17. 
  
\end{list2}


\section{\sc 2017}
\begin{list2}
\item [{\bf 9}.] Genomic connectedness across management units. The 62nd RBras (The Brazilian Region of the International Biometric Society) and 17th SEAGRO (Symposium on Statistics Applied to Agricultural Experimentation) Meeting. Federal University of Lavras, Lavras, MG, Brazil. July 24-28. 
  
  \vspace{0.5cm}
  
  \item [{\bf 8}.] Applications of data mining and prediction methods to animal sciences. Symposium on Big Data Analytics and Precision Animal Agriculture. ASAS-CSAS Annual Meeting. Baltimore Convention Center, Baltimore, MD. July 8-12. 
  
\end{list2}



\section{\sc 2016}
\begin{list2}
\item [{\bf 7}.] Phenome-wide genetic mean effect and variance heterogeneity association analysis. Biological Sciences Graduate Seminar. School of Biological Sciences, University of Nebraska-Lincoln, Lincoln, NE. September 23.
  
  \vspace{0.5cm}
  
  \item [{\bf 6}.] MeSH annotation of the chicken genome. Poultry 2 Workshop. The Plant and Animal Genome XXIV Conference. Town and Country Hotel, San Diego, CA. January 9-13. 
  
\end{list2}

\section{\sc 2015}
\begin{list1}
\item [{\bf 5}.] Inferring the impact of population stratification on genomic heritability using a reparameterized genomic best linear unbiased prediction model. Statistics Seminars. Department of Statistics, University of Nebraska-Lincoln, Lincoln, NE. September 23. 

  \vspace{0.5cm}

\item [{\bf 4}.] Quantitative genetics in the functional genomics era. Animal Breeding \& Genetics Seminars. Department of Animal Science, Iowa State University, Ames, IA. March 3. 
\end{list1}

\section{\sc 2014}
\begin{list1}
\item [{\bf 3}.] Quantitative genetics in the functional genomics era. Special Seminar. PIC, Inc., Hendersonville, TN.  November 12. 
\end{list1}

\section{\sc 2013}
\begin{list1}
\item [{\bf 2}.] Whole-genome prediction of complex traits using kernel methods. Department of Animal Science, University of Nebraska-Lincoln, Lincoln, NE. December 19. 
\end{list1}

\section{\sc 2011}
\begin{list1}
\item [{\bf 1}.] Obihiro GCOE Animal Global Health Seminars.  Obihiro University of Agriculture and  Veterinary Medicine, Obihiro, Hokkaido, Japan. January 7. 
\end{list1}  



\vspace{0.5cm}
%\section{\sc Contributed Abstracts}
\section{\sc Contributed Presentations}
\vspace{1.0cm}



\section{\sc 2018}
\begin{list2}

\item [{\bf 15}.] Longitudinal genomic prediction of image-derived phenotypes and interactive visualization tools. Special seminar. Breeding Unit, Division of Apple Research, Institute of Fruit Tree and Tea Science, Shimo-kuriyagawa, Morioka, Iwate, Japan. November 22.
  
  \vspace{0.5cm}
  
\item [{\bf 14}.] Multivariate analyses for longitudinal phenotypes and genome-wide association studies in plant and animals. Special seminar. Crop Science Laboratory, Faculty of Agriculture, Iwate University, Ueda, Morioka, Iwate, Japan. November 21.
  
  \vspace{0.5cm}

\item [{\bf 13}.] Longitudinal genomic prediction of image-derived phenotypes in rice using a random regression model. The 8th Rice Genetics Symposium (RG8), The International Rice Research Conference 2018 (IRRC 2018). Marina Bay Sands, Singapore. October 15-17.
  
  \vspace{0.5cm}
  
\item [{\bf 12}.] Genome-enabled prediction and genome-wide association analysis for longitudinal image-based data in rice. The 134th Japanese Society of Breeding Meeting. Okayama University, Kita Ward, Okayama, Japan. September 22-23.
  
  \vspace{0.5cm}

  
  \item [{\bf 11}.] Investigating the relationship between microbial community and carcass traits in beef cattle.  The 124th Japanese Society of Animal Science Meeting. The University of Tokyo, Bunkyo-ku, Tokyo, Japan. March 27-30.

    
    \vspace{0.5cm}

          
\item [{\bf 10}.] Stronger measures of genomic connectedness enhance prediction accuracies across management units. The 11th World Congress of Genetics Applied to Livestock Production.  Aotea Centre, Auckland, New Zealand. February 11-16. 

\end{list2}  





\section{\sc 2017}
\begin{list2}

  \item [{\bf 9}.] ShinyGPAS: Interactive genomic prediction accuracy simulator based on deterministic formulas. NCERA-225 Meeting. Stanley Stout Livestock Marketing Center, Manhattan, KS. October 18-19. 

      \vspace{0.5cm}

  \item [{\bf 8}.] Genomic connectedness across management units. The 123rd Japanese Society of Animal Science Meeting. Shinshu University, Kamiina, Nagano, Japan. September 4-8.

\end{list2}  

    
\section{\sc 2015}
\begin{list2}

  \item [{\bf 7}.] Quantitative genetics in the functional genomics era. Special Seminar. The National Institute of Agrobiological Sciences, Tsukuba, Japan. November 12. 

  \vspace{0.5cm}

\item [{\bf 6}.] Quantitative genetics in the functional genomics era. Special Seminar. Laboratory of Biometry and Bioinformatics, The University of Tokyo, Bunkyo-ku, Tokyo, Japan. November 6. 

  \vspace{0.5cm}
  
\item [{\bf 5}.] The impact of population stratification on genomic heritability. NCERA-225 Meeting. North Dakota State University, Fargo, ND. October 22-23. 

  \vspace{0.5cm}

\item [{\bf 4}.] An application of MeSH enrichment analysis in livestock. ADSA-ASAS Joint Annual Meeting. Rosen Shingle Creek, Orlando FL. July 12-16.  

  \vspace{0.5cm}
  
\item [{\bf 3}.] Prediction of complex quantitative traits using functional annotations and bootstrap aggregating. Special Seminar. National Livestock Breeding Center, Shirakawa, Japan. January 10. 
\end{list2}  

\section{\sc 2012}
\begin{list2}
\item  [{\bf 2}.] Application of Bayesian and Sparse Network Models for Assessing Linkage Disequilibrium in Animals and Plants. 26th International Biometric Conference. Kobe International Conference Center, Kobe Japan. August 26-31.
  \textcolor{blue}{\href{http://secretariat.ne.jp/ibc2012/30Aug.html\#aug-30-14:00-Contributed36}{http://secretariat.ne.jp/ibc2012/30Aug.html\#aug-30-14:00-Contributed36. } }   {\bf $\star$Second Oral Prize Winners}.
\end{list2}


\section{\sc 2007}
\begin{list2}
\item  [{\bf 1}.] The impact of missing information in continuous and threshold trait analyses under a linear mixed model framework. The 62nd Hokkaido Animal Science and Agriculture Society Meeting. Obihiro University of Agriculture and Veterinary Medicine, Obihiro, Hokkaido, Japan. September 5-6.
\end{list2}


\vspace{0.5cm}
\section{ \sc Posters }
\vspace{0.5cm}
\section{\sc 2015}
\begin{list2}

\item Population stratification contribution to genomic heritability. Probabilistic Modeling in Genomics. Cold Spring Harbor Laboratory, NY. October 14 - 17. 

  \vspace{0.5cm}
  
\item Estimating genomic heritability in the presence of population stratification. NGS Field 4th Meeting. Tsukuba International Congress Center, Tsukuba, Japan. July 1-3. 
\end{list2}

\section{\sc 2013}
\begin{list2}
\item   MeSHR: R/Bioconductor package for finding statistically overrepresented  MeSH terms in a set of genes. Annual Bioconductor Conference BioC 2013. July 18-19, Seattle, WA. \\ \textcolor{blue}{\href{https://secure.bioconductor.org/BioC2013/posters.php\#7}{https://secure.bioconductor.org/BioC2013/posters.php\#8. } }  

\vspace{0.5cm}

\item Predicting complex traits using a diffusion kernel on genetic markers with an application to dairy cattle and wheat data. Annual Bioconductor Conference BioC 2013. July 18-19, Seattle, WA. \textcolor{blue}{\href{https://secure.bioconductor.org/BioC2013/posters.php\#7}{https://secure.bioconductor.org/BioC2013/posters.php\#7. } }  
\end{list2}








%\section{SCIENCE OUTREACH & VOLUNTEER}



\vspace{0.5cm}
\section{\sc Intramural Seminars}
\vspace{1cm}
\section{\sc 2019}
\begin{list2}
  \item Translational Plant Sciences Discussion Group. Virginia Polytechnic Institute and State University, Blacksburg, VA. September 26. 
  \vspace{0.25cm}
\item Reproductive Biology Club. Virginia Polytechnic Institute and State University, Blacksburg, VA. April 19. 
\end{list2}  
\section{\sc 2017}
\begin{list2}
\item Monthly Brown Bag Series on Plant Phenotyping. University of Nebraska-Lincoln, Lincoln, NE. March 31. 
\end{list2}  
\section{\sc 2015}
\begin{list2}
\item  Animal Breeding \& Genetics Seminars. Department of Animal Science, University of Nebraska-Lincoln, Lincoln, NE. September 15. 
\end{list2}  
\section{\sc 2014}
\begin{list2}
\item  Animal Breeding \& Genetics Seminars. Department of Animal Science, University of Nebraska-Lincoln, Lincoln, NE. September 18. 
\vspace{0.25cm}
\item  Ph.D. Thesis Defense. Department of Animal Sciences, University of Wisconsin-Madison, Madison, WI. May 12. 
\vspace{0.25cm}
\item  Dairy Science Graduate Seminars. Department of Dairy Science, University of Wisconsin-Madison, Madison, WI. February 14. 
\end{list2}  
\section{\sc 2013}
\begin{list2}
\item  Special Seminar.  Zoetis, Inc., Kalamazoo, MI. August 8. 
\end{list2}  
\section{\sc 2012}
\begin{list2}
 \item Animal Breeding \& Genomics Seminars.  Department of Animal Sciences, University of Wisconsin-Madison, Madison, WI. April 10.
\end{list2}  
\section{\sc 2011}
\begin{list2}
\item  Master's Thesis Defense.  Department of Dairy Science, University of Wisconsin-Madison, Madison, WI. December 5. 
\end{list2}  
\section{\sc 2010}
\begin{list2}
\item Animal Breeding \& Genomics Seminars.  Department of Dairy Science, University of Wisconsin-Madison, Madison, WI. November 30.
\vspace{0.25cm}
\item Animal Breeding \& Genomics Seminars.  Department of Dairy Science, University of Wisconsin-Madison, Madison, WI. March 23. 
\end{list2}
\section{\sc 2008}
\begin{list2}
\item  Animal Breeding \& Genomics Seminars.  Department of Dairy Science, University of Wisconsin-Madison, Madison, WI. November 25. 
\end{list2}




\vspace{0.5cm}
\section{\sc Teaching}
{\bf Virginia Polytechnic Institute and State University}, Blacksburg, Virginia, USA  \vspace{0.2cm} \\
\underline{Lead Instructor}
\vspace{0.2cm}
\begin{list2}

\item GRAD 5515 Molecular Plant Science Laboratory Rotation
  \hfill {\bf Fall, 2019} 

\end{list2}


{\bf University of Nebraska-Lincoln}, Lincoln, Nebraska, USA  \vspace{0.2cm} \\
\underline{Lead Instructor}
\vspace{0.2cm}
\begin{list2}

\item ASCI 944 / STAT 844 Quantitative Methods for Genomics of Complex Traits
  \hfill {\bf Spring, 2018} 
   [\textcolor{blue}{\href{http://morotalab.org/asci944-2018/ASCI944.html}{WWW}}]

\item ASCI 896 Statistical Genomics [\textcolor{blue}{\href{http://morotalab.org/asci896-2017/ASCI896.html}{WWW}}]
  \hfill {\bf Spring, 2017}
  %11 students  % 8 + 3

\item ASCI 896 Statistical Genomics [\textcolor{blue}{\href{http://morotalab.org/asci896-2016/ASCI896.html}{WWW}}]
  \hfill {\bf Spring, 2016} 
%14 students  % 11 + 3

\end{list2}
\vspace{.01pt}
        

\underline{Co-Instructor} 
\vspace{0.2cm}
\begin{list2}
  
\item STAT 892-004 Integrative Data Science for Plant Phenomics [\textcolor{blue}{\href{http://morotalab.org/stat892-2018/STAT892.html}{WWW}}]
  \hfill {\bf Spring, 2018}
  
  
\item ASCI 431/831 Advanced Animal Breeding [\textcolor{blue}{\href{http://morotalab.org/asci431-2018/ASCI431-831.html}{WWW}}]
  \hfill {\bf Spring, 2018} 


\item LIFE 891-002 Integrating Quantitative and Computational Biology into Life Sciences Research [\textcolor{blue}{\href{http://morotalab.org/life431-2018/life431-831.html}{WWW}}]
  \hfill {\bf Spring, 2018} 


\item ASCI 431/831 Advanced Animal Breeding [\textcolor{blue}{\href{http://morotalab.org/asci431-2017/ASCI431-831.html}{WWW}}]
  \hfill {\bf Spring, 2017} 

\end{list2}

 \vspace{.01pt}

\underline{Guest Instructor}
 \vspace{0.2cm}
 \begin{list2}
 \item ASCI 432/832 Genome Analysis - April 21  \hfill {\bf Spring, 2017}
\vspace{.01pt}
\item ASCI/AGRO 931  Population Genetics - November 2  \hfill {\bf Fall, 2016}
\vspace{.01pt}
\item ASCI 432/832 Genome Analysis - April 15  \hfill {\bf Spring, 2016}
\vspace{.01pt}
\item ASCI 432/832 Genome Analysis - April 16    \hfill {\bf Spring, 2015}
\end{list2}

 
{\bf The University of Tokyo}, Bunkyo-ku, Tokyo, Japan \vspace{0.2cm} \\
\underline{Guest Instructor}
\vspace{0.2cm}
 \begin{list2}
 \item 060310391/0560565 Biometrics - November 26 \hfill {\bf Fall, 2018}
 \end{list2}

 
{\bf University of Wisconsin-Madison}, Madison, Wisconsin, USA  \vspace{0.2cm}  \\
\underline{Teaching Assistant} 
 \vspace{0.2cm}
\begin{list2}
\item ANSCI/DYSCI 363: Principles of Animal Breeding    \hfill {\bf Spring, 2011}

\vspace{.01pt}
\item ANSCI/DYSCI 361: Introduction to Animal and Veterinary Genetics    \hfill {\bf Spring, 2011} 

  \end{list2}



\vspace{0.5cm}
\section{\sc Short Courses}
%jointly with Drs.


{\bf The 64th RBras and 18th SEAGRO Meeting}, Cuiab{\'a}, MT, Brazil.
\vspace{.01pt}

Lead Instructor  \\
    Quantitative Genetics Short Courses - [\textcolor{blue}{\href{http://morotalab.org/RBras2019/RBras2019.html}{WWW}}]
\hfill {\bf  July 29 - August 2, 2019} \\
20 participants 

    
{\bf The Hebrew University of Jerusalem}, Rehovot, Israel
\vspace{.01pt}

Co-Instructor  \\
Bridging the Gap: From Phenomics to Functional Genetics - [\textcolor{blue}{\href{http://morotalab.org/HUJI2019/HUJI2019.html}{WWW}}]
\hfill {\bf April 1-3, 2019} \\
20 participants 


{\bf University of S\~{a}o Paulo / ESALQ}, Piracicaba, S\~{a}o Paulo, Brazil
\vspace{.01pt}

Co-Instructor  \\
Quantitative Genetics and Genomics Workshop - [\textcolor{blue}{\href{http://morotalab.org/ESALQ2019/ESALQ2019.html}{WWW}}]
\hfill {\bf May 20-24, 2019} \\
20 participants 


Co-Instructor  \\
Quantitative Genetics and Genomics Workshop - [\textcolor{blue}{\href{http://morotalab.org/ESALQ2018/ESALQ2018.html}{WWW}}]
\hfill {\bf May 21-25, 2018} \\
55 participants 


Co-Instructor  \\
Quantitative Genetics and Genomics Workshop - [\textcolor{blue}{\href{http://morotalab.org/ESALQ2016/ESALQ2016.html}{WWW}}]
\hfill {\bf May 16-20, 2016} \\
35 participants  


{\bf Federal University of Vi\c cosa}, Vi\c cosa, MG, Brazil
\vspace{.01pt}

Lead Instructor  \\
Quantitative Genetics Workshop - [\textcolor{blue}{\href{http://morotalab.org/UFV2019/UFV2019.html}{WWW}}]
\hfill {\bf November 18-26, 2019} \\
15 participants 


Co-Instructor  \\
Linear Mixed Model Workshop - [\textcolor{blue}{\href{http://morotalab.org/UFV2018/UFV2018.html}{WWW}}]
\hfill {\bf October 26, 2018} \\
20 participants 

{\bf The University of Tokyo}, Bunkyo-ku, Tokyo, Japan
\vspace{.01pt}

Co-Instructor  \\
Statistical Methods for Omics-assisted Breeding Workshop - [\textcolor{blue}{\href{http://morotalab.org/UT2018/UT2018.html}{WWW}}]
\hfill {\bf November 12-15, 2018} \\
50 participants 


{\bf Virginia Polytechnic Institute and State University}, Blacksburg, VA, USA
\vspace{.01pt}

Co-Instructor  \\
GWAS Workshop - [\textcolor{blue}{\href{http://morotalab.org/VTGWAS2019/VTGWAS2019.html}{WWW}}]
\hfill {\bf June 24-26, 2019} \\
20 participants 


\vspace{0.5cm}
\section{\sc Research Support}
\begin{flushleft}
\hspace{0.2cm} \underline{External Funding}
\end{flushleft}

\begin{list2}


\item Exploratory Research Program - \$200,000.00  \hfill USDA-NIFA\\
  PI: Kiho Lee   \hfill \textbf{October 1, 2019 - September 30, 2021}\\
Proposal: Wireless monitoring and assess system to improve productivity and animal welfare in swine  \\
Role: Co-Principal Investigator \\


\vspace{0.5cm}


\item Food Safety Challenge Area: Effective Mitigation Strategies for Antimicrobial Resistance  - \$773,607.00  \hfill USDA-NIFA\\
PI: Samodha Fernando   \hfill \textbf{February 15, 2018 - February 14, 2022}\\
Proposal: Investigating mobile genetic elements and resistance gene reservoirs towards understanding the emergence and ecology of antimicrobial resistance in beef cattle production systems  \\
Role: Co-Principal Investigator \\

\vspace{0.5cm}

\item Animal Health and Production and Animal Products: Improved Nutritional Performance, Growth, and Lactation of Animals  - \$500,000.00  \hfill USDA-NIFA\\
PI: Samodha Fernando   \hfill \textbf{March 1, 2018 - February 28, 2022}\\
Proposal: Moving beyond rumen microbiota composition to identify interactions between host genotype and rumen function towards identifying genetic markers and microbial functions that influence feed efficiency  \\
Role: Co-Principal Investigator \\

\vspace{0.5cm}

%\item NEDED  - \$21,472.00  \hfill NuGUT LLC\\
%PI: Gota Morota   \hfill \textbf{September 1, 2017 - August 31, 2018}\\
%Proposal: Development of Statistical Software for Detecting Fetal Aneuploidy in Livestock Species  \\
%Role: Principal Investigator \\

\vspace{0.5cm}
  
\item EPSCoR Research Infrastructure Improvement Program - \$5,783,738.00   \hfill NSF \\
PI: Harkamal Walia   \hfill \textbf{August 1, 2017 - July 31, 2021}\\
Proposal:  Comparative genomics and phenomics approach to discover genes underlying heat stress resilience in cereals  (RII Track-2 FEC)  \\
Award number: 1736192 \\
Role: Co-Principal Investigator \\

\end{list2}




\begin{flushleft}
\hspace{0.2cm} \underline{Internal Funding}
\end{flushleft}
\begin{list2}

\item SmartFarm Innovation Network \$349,150.00  \hfill VT\\
PI: Robin White / Vitor Mercadante  \hfill \textbf{October 1, 2019 - September 30, 2021}\\
Proposal: Establishment of SmartFarm innovation network nodes at Middleburg and Shenandoah Valley Agricultural Research and Extension Centers \\
Role: Co-Principal Investigator \\

\vspace{0.5cm}


\item  Hebrew University of Jerusalem - Virginia Tech Joint Travel Grant \$500.00  \hfill HUJI-VT\\
PI: Zvi Peleg  \hfill \textbf{August 25, 2019 - August 30, 2019}\\
Proposal: Deciphering the genetic architecture of wheat root system \\
Role: Co-Principal Investigator \\


\vspace{0.5cm}



\item ICAT SEAD Grant \$25,000.00  \hfill VT\\
PI: Koeun Choi  \hfill \textbf{July 15, 2019 - June 30, 2020}\\
Proposal:  Mobile learning across the life span: Processing and learning information from mobile media technology in children, young adults, and older adults \\
Role: Co-Principal Investigator \\


\vspace{0.5cm}

\item New Faculty Mentoring Project Grant  - \$1,500.00  \hfill VT\\
PI: Gota Morota   \hfill \textbf{January 11, 2020 - January 15, 2020}\\
Proposal: Participating in the Plant \& Animal Genome Conference XXVIII\\
Role: Principal Investigator \\


\vspace{0.5cm}

\item IANR Travel Funds  - \$800.00  \hfill UNL\\
PI: Gota Morota   \hfill \textbf{February 11, 2018 - February 16, 2018}\\
Proposal: Participating in the World Congress on Genetics Applied to Livestock Production  \\
Role: Principal Investigator \\


\vspace{0.5cm}

\item SPRINT 4th Edition  - \$18,300.00  \hfill UNL/FAPESP\\
PI: Gota Morota   \hfill \textbf{June 1, 2017 - May 31, 2019}\\
Proposal: Integration of genomic resources in beef cattle breeding program - a collaborative effort between UNL and ESALQ  \\
Role: Principal Investigator \\


\vspace{0.5cm}
  
\item ARD Plant Phenotyping Seed Grant  - \$100,000.00  \hfill UNL\\
PI: Gota Morota   \hfill \textbf{January 1, 2017 - June 30, 2018}\\
Proposal: Development of imaging-informed dynamic subgenome specific co-expression gene networks in wheat  \\
Role: Principal Investigator \\

\vspace{0.5cm}

\item Research Council Interdisciplinary Grant  - \$20,000.00  \hfill UNL\\
PI: Gota Morota   \hfill \textbf{January 1, 2017 - December 31, 2017}\\
Proposal: Advancing plant phenomics through leveraging an image-based longitudinal quantitative genetics model and a gene annotation tool  \\
Role: Principal Investigator \\

\vspace{0.5cm}


\item IANR International Impact Award - \$3,000.00  \hfill UNL\\
PI: Gota Morota   \hfill \textbf{May 16, 2016 - May 20, 2016}\\
Proposal: Delivering a graduate training program at University of S{\~a}o Paulo / ESALQ  \\
Role: Principal Investigator \\ 

\vspace{0.5cm}

\item ORED Layman Seed Award - \$9,910.00  \hfill UNL\\
PI: Gota Morota   \hfill \textbf{June 1, 2015 - May 31, 2016}\\
Proposal: Cracking the blackbox of whole-genome prediction: Genome partitioning of predictive ability \\
Role: Principal Investigator \\


\end{list2}



\vspace{0.5cm}
\section{\sc Advisees and trainees}

\begin{flushleft}
\hspace{0.2cm} \underline{Postdoctoral Scholars}
\end{flushleft}
\begin{enumerate}
  
\item Mehdi Momen [\textcolor{blue}{\href{https://mehdimomen.github.io/}{WWW}}]  \hfill 11/27/2018 - 11/26/2019
  \begin{itemize} 
  \item Current position: Postdoctoral Scholar, University of Wisconsin-Madison 
  \end{itemize}
  
  \vspace{0.3cm}
  
\item Waseem Hussain  [\textcolor{blue}{\href{https://whussain2.github.io/}{WWW}}]  \hfill 3/9/2018 - 7/26/2019
  \begin{itemize} 
  \item Current position: Research Scientist, International Rice Research Institute
  \end{itemize}
   
  \vspace{0.3cm}
  
\item Malachy T. Campbell  [\textcolor{blue}{\href{https://malachycampbell.github.io/}{WWW}}]  \hfill  9/1/2017 - 9/30/2019
  \begin{itemize} 
  \item Current position: Postdoctoral Scholar, Cornell University 
  \end{itemize}
\end{enumerate}



\begin{flushleft}
\hspace{0.2cm} \underline{Ph.D. Students}
\end{flushleft}
\begin{enumerate}

  \item Idan Sabag (jointly with Zvi Peleg) [\textcolor{blue}{\href{}{WWW}}]  \hfill  10/24/2019 -

    \vspace{0.3cm}

\item Haipeng Yu [\textcolor{blue}{\href{https://haipengu.github.io/}{WWW}}]  \hfill 8/22/2016 -
\end{enumerate}








\begin{flushleft}
\hspace{0.2cm} \underline{Visiting Scholars}
\end{flushleft}
\begin{enumerate}

\item Luiz A. Peternelli, Federal University of Vi\c cosa [\textcolor{blue}{\href{http://www.dpi.ufv.br/~peternelli/}{WWW}}]  \hfill 12/9/2019 - 2/28/2020

  \vspace{0.3cm}

      
\item Toshimi Baba, Hokkaido Holstein Agricultural Association [\textcolor{blue}{\href{https://researchmap.jp/t-baba/?lang=english}{WWW}}]  \hfill  4/22/2019 -
  
  \vspace{0.3cm}

\item Jun He, Hunan Agricultural University (jointly with Matt Spangler \& Steve Kachman)  \hfill  8/2015 - 2/2016 
\end{enumerate}



\begin{flushleft}
\hspace{0.2cm} \underline{Visiting Postdoctoral Scholars}
\end{flushleft}
\begin{enumerate}
\item Sara Pegolo,  University of Padova [\textcolor{blue}{\href{https://www.researchgate.net/profile/Sara_Pegolo}{WWW}}]  \hfill 1/21/2019 - 2/1/2019
  
  \vspace{0.3cm}

\item Juliana Petrini,  University of Sao Paulo  [\textcolor{blue}{\href{https://www.researchgate.net/profile/Juliana_Petrini}{WWW}}]  \hfill  4/16/2018 - 5/4/2018
\end{enumerate}






\begin{flushleft}
\hspace{0.2cm} \underline{Visiting Ph.D. Students}
\end{flushleft}
\begin{enumerate}
\item Francisco Jos\'{e} de Novais,  University of Sao Paulo [\textcolor{blue}{\href{https://fjnovais.github.io/}{WWW}}]  \hfill  9/3/2019 - 2/29/2020

\vspace{0.3cm}

\item Gerardo Mamani, University of Sao Paulo  [\textcolor{blue}{\href{https://github.com/gerardocorn}{WWW}}]  \hfill 4/12/2017 - 12/31/2017
\end{enumerate}



\begin{flushleft}
\hspace{0.2cm} \underline{Visiting M.S. Students}
\end{flushleft}
\begin{enumerate}
\item Sabrina T. Amorim, Sao Paulo State University [\textcolor{blue}{\href{https://sabrinaam.github.io/}{WWW}}]  \hfill 5/28/2019 - 11/27/2019
\end{enumerate}


\vspace{0.5cm}
\section{\sc Thesis Committees}

\begin{flushleft}
\hspace{0.2cm} \underline{Ph.D Thesis Committees}
\end{flushleft}
\begin{enumerate}
\item Amanda B. Alvarenga   \hfill 2019 - \\
  Department of Animal Sciences, Purdue University \\
  Major advisor: Luiz F. Brito
\end{enumerate}


\begin{flushleft}
\hspace{0.2cm} \underline{M.S. Thesis Committees}
\end{flushleft}
\begin{enumerate}
\item Mateus Teles Vital Gon\c calves  \hfill July 2019 \\
  Genetics and Plant Breeding Program, Federal University of  Vi\c cosa\\
  Major advisor: Luiz A. Peternelli
\end{enumerate}




\vspace{0.5cm}
\section{\sc Service Activities}

\begin{flushleft}
\hspace{0.2cm} \underline{Visitors hosted}
\end{flushleft}
\begin{list2}
\item Daniel Gianola, University of Wisconsin-Madison  \hfill September 2019\\

\item Zvi Peleg, The Hebrew University of Jerusalem   \hfill August 2019\\

\item Yutaka Masuda, University of Georgia \hfill April 2019\\


\item Luiz A. Peternelli, Federal University of Vi\c cosa  \hfill July 2018\\

    
\item Luiz L. Coutinho, University of Sao Paulo / ESALQ  \hfill August 2017\\


\end{list2}


\begin{flushleft}
\hspace{0.3cm} \underline{Multistate research activities}
\end{flushleft}
\begin{list2}
%\item NRSP-8 Swine: ational Animal Genome Research Program \\ Virginia Polytechnic Institute and State University representative \hfill  2019 - Present\\

 %   \vspace{0.3cm}

\item NCERA-225: Implementation and Strategies for National Beef Cattle Genetic Evaluation \\ University of Nebraska-Lincoln representative \hfill  2015 - 2018\\
\end{list2}




\begin{flushleft}
\hspace{0.3cm} \underline{Ad hoc Review of Proposals}
\end{flushleft}
\begin{list2}
\item BBSRC grant proposal reviewer (2014)
\end{list2}

%International and National
% University
\begin{flushleft}
  \hspace{0.3cm} \underline{Departmental}
\end{flushleft}
%\vspace{0.5cm}
\begin{list2}
\item Graduate Programs Committee \\
  Department of Animal and Poultry Sciences \\ Virginia Polytechnic Institute and State University   \hfill \textbf{2019-2021}\\

\item Research Programs Committee \\
  Department of Animal and Poultry Sciences \\ Virginia Polytechnic Institute and State University  \hfill \textbf{2019-2021}\\

\end{list2}


\begin{flushleft}
\hspace{0.3cm} \underline{Research Area}
\end{flushleft}
%\vspace{0.5cm}
\begin{list2}
\item Animal Breeding \& Genetics Seminars organizer \\
  Department of Animal Science, University of Nebraska-Lincoln   \hfill \textbf{Spring 2016}\\

\item Animal Breeding \& Genetics Seminars organizer \\
  Department of Animal Science, University of Nebraska-Lincoln   \hfill \textbf{Fall 2015}\\
\end{list2}






\vspace{0.5cm}
\section{\sc OSS contributions} 
\begin{list1}
\item[] R packages
\begin{list2}
\item dkDNA - \textcolor{blue}{\href{http://cran.r-project.org/web/packages/dkDNA/index.html}{http://cran.r-project.org/web/packages/dkDNA/index.html}}
\end{list2}


\vspace{0.3cm}
\item[] Shiny Applications
  \begin{list2}
  \item ShinyAIM - \textcolor{blue}{\href{https://chikudaisei.shinyapps.io/shinyaim/}{https://chikudaisei.shinyapps.io/shinyaim/}}

    \vspace{0.3cm}
    
\item ShinyGPAS - \textcolor{blue}{\href{https://chikudaisei.shinyapps.io/shinygpas/}{https://chikudaisei.shinyapps.io/shinygpas/}} 
\end{list2}



\vspace{0.3cm}
\item[] Bioconductor packages
\begin{list2}
\item \textcolor{blue}{\href{http://bioconductor.org/packages/release/bioc/html/meshr.html}{meshr}}
\end{list2}
\vspace{0.3cm}
\begin{list2}
\item  \textcolor{blue}{\href{http://bioconductor.org/packages/release/data/annotation/html/MeSH.db.html}{MeSH.db}}
\end{list2}
\vspace{0.3cm}
\begin{list2}
\item \textcolor{blue}{\href{http://www.bioconductor.org/packages/release/data/annotation/html/MeSH.AOR.db.html}{MeSH.AOR.db}}
\end{list2}
\vspace{0.3cm}
\begin{list2}
\item \textcolor{blue}{\href{http://www.bioconductor.org/packages/release/data/annotation/html/MeSH.PCR.db.html}{MeSH.PCR.db}}
\end{list2}
\vspace{0.3cm}
\begin{list2}
\item \textcolor{blue}{\href{http://www.bioconductor.org/packages/release/data/annotation/}{MeSH.XXX.eg.db }} (84 packages)
\end{list2}
\begin{multicols}{2}
%\interlinepenalty=100000
\begin{itemize}
\item MeSH.Aca.eg.db
\item MeSH.Aga.PEST.eg.db
\item MeSH.Ame.eg.db
\item MeSH.Aml.eg.db
\item MeSH.Ana.eg.db
\item MeSH.Ani.FGSC.eg.db
\item MeSH.Ath.eg.db
%\item MeSH.Atu.K84.eg.db
\item MeSH.Bfl.eg.db
\item MeSH.Bsu.168.eg.db
%\item MeSH.Bsu.Bsn5.eg.db 
%\item MeSH.Bsu.RONN1.eg.db
\item MeSH.Bsu.TUB10.eg.db
%\item MeSH.Bsu.W23.eg.db  
\item MeSH.Bta.eg.db 
\item MeSH.Cal.SC5314.eg.db
\item MeSH.Cbr.eg.db 
\item MeSH.Cel.eg.db
\item MeSH.Cfa.eg.db
\item MeSH.Cin.eg.db
\item MeSH.Cja.eg.db
\item MeSH.Cpo.eg.db
\item MeSH.Cre.eg.db  
\item MeSH.Dan.eg.db
\item MeSH.Dda.3937.eg.db
\item MeSH.Ddi.AX4.eg.db
\item MeSH.Der.eg.db 
\item MeSH.Dgr.eg.db 
\item MeSH.Dme.eg.db 
\item MeSH.Dmo.eg.db 
\item MeSH.Dpe.eg.db
\item MeSH.Dre.eg.db
\item MeSH.Dse.eg.db
\item MeSH.Dsi.eg.db
\item MeSH.Dvi.eg.db  
\item MeSH.Dya.eg.db
%\item MeSH.Eco.536.eg.db    
\item MeSH.Eco.55989.eg.db
%\item MeSH.Eco.APEC01.eg.db 
%\item MeSH.Eco.B.REL606.eg.db
%\item MeSH.Eco.BW2952.eg.db 
\item MeSH.Eco.CFT073.eg.db
%\item MeSH.Eco.E24377A.eg.db   
\item MeSH.Eco.ED1a.eg.db 
\item MeSH.Eco.HS.eg.db
\item MeSH.Eco.IAI1.eg.db 
\item MeSH.Eco.IAI39.eg.db 
\item MeSH.Eco.K12.DH10B.eg.db
\item MeSH.Eco.K12.MG1655.eg.db
%\item MeSH.Eco.KO11FL.eg.db 
%\item MeSH.Eco.O103.H2.12009.eg.db
%\item MeSH.Eco.O111.H.11128.eg.db 
\item MeSH.Eco.O127.H6.E2348.69.eg.db
%\item MeSH.Eco.O157.H7.EC4115.eg.db
\item MeSH.Eco.O157.H7.EDL933.eg.db 
\item MeSH.Eco.O157.H7.Sakai.eg.db
%\item MeSH.Eco.O157.H7.TW14359.eg.db
%\item MeSH.Eco.O26.H11.11368.eg.db 
%\item MeSH.Eco.O26.H7.CB9615.eg.db 
\item MeSH.Eco.S88.eg.db
%\item MeSH.Eco.SE11.eg.db 
%\item MeSH.Eco.SMS35.eg.db 
\item MeSH.Eco.UMN026.eg.db
%\item MeSH.Eco.UTI89.eg.db 
\item MeSH.Eqc.eg.db 
\item MeSH.Gga.eg.db
\item MeSH.Gma.eg.db 
\item MeSH.Hsa.eg.db
\item MeSH.Laf.eg.db 
\item MeSH.Lma.eg.db 
\item MeSH.Mdo.eg.db 
\item MeSH.Mes.eg.db 
\item MeSH.Mga.eg.db 
\item MeSH.Miy.eg.db 
\item MeSH.Mml.eg.db 
\item MeSH.Mmu.eg.db 
\item MeSH.Mtr.eg.db 
\item MeSH.Nle.eg.db 
\item MeSH.Oan.eg.db 
\item MeSH.Ocu.eg.db 
\item MeSH.Oni.eg.db 
\item MeSH.Osa.eg.db 
\item MeSH.Pab.eg.db 
%\item MeSH.Pae.LESB58.eg.db 
%\item MeSH.Pae.PA14.eg.db
%\item MeSH.Pae.PA7.eg.db
\item MeSH.Pae.PAO1.eg.db 
\item MeSH.Pfa.3D7.eg.db
\item MeSH.Pto.eg.db 
\item MeSH.Ptr.eg.db 
\item MeSH.Rno.eg.db  
%\item MeSH.Sau.COL.eg.db 
%\item MeSH.Sau.ED98.eg.db
%\item MeSH.Sau.M013.eg.db 
%\item MeSH.Sau.MRSA252.eg.db 
%\item MeSH.Sau.MSHR1132.eg.db
%\item MeSH.Sau.MSSA476.eg.db
%\item MeSH.Sau.Mu3.eg.db
%\item MeSH.Sau.Mu50.eg.db 
%\item MeSH.Sau.MW2.eg.db  
%\item MeSH.Sau.N315.eg.db 
%\item MeSH.Sau.Newman.eg.db 
%\item MeSH.Sau.RF122.eg.db 
%\item MeSH.Sau.USA300FPR3757.eg.db 
\item MeSH.Sau.USA300TCH1516.eg.db 
%\item MeSH.Sau.VC40.eg.db
\item MeSH.Sce.S288c.eg.db 
\item MeSH.Sco.A32.eg.db 
\item MeSH.Sil.eg.db 
\item MeSH.Spo.972h.eg.db
\item MeSH.Spu.eg.db
\item MeSH.Ssc.eg.db 
\item MeSH.Syn.eg.db 
\item MeSH.Tbr.9274.eg.db 
\item MeSH.Tgo.ME49.eg.db
\item MeSH.Tgu.eg.db 
\item MeSH.Vvi.eg.db 
\item MeSH.Xla.eg.db  
\item MeSH.Xtr.eg.db 
\item MeSH.Zma.eg.db
\end{itemize}
\end{multicols}



\vspace{0.3cm}
\item[] Github
\begin{list2}
\item \textcolor{blue}{\href{https://github.com/morota}{https://github.com/morota}} 
\end{list2}
\end{list1}









\vspace{0.5cm}
\section{\sc Participation in meetings, symposiums, and workshops} 
\vspace{2cm}


\section{\sc 2019}
\begin{list2}

\item NCERA-225 Meeting. Implementation and Strategies for National Beef Cattle Genetic Evaluation. Alphin Stuart Livestock Arena, Blacksburg, VA. October 10-11.

  \vspace{0.5cm}

\item Phenome 2019. El Conquistador Tucson, A Hilton Resort, Tucson, AZ. February 6-9.

\end{list2}




\section{\sc 2018}
\begin{list2}

  \item Agrigenomic Industry Workshop. Co-working space Kayabacho Co-Edo, Chuo-ku, Tokyo, Japan. September 14

    \vspace{0.5cm}
    
\item UNL Plant Phenomics Symposium. Cather Dining Complex, University of Nebraska-Lincoln, Lincoln, NE. April 2.

\end{list2}


\section{\sc 2017}
\begin{list2}

\item EPSCoR 2017 Track 2 Kickoff Meeting.  National Science Foundation, Alexandria, VA. October 3.
  
  \vspace{0.5cm}

\item The 15th International Symposium on Rice Functional Genomics. Gyeonggi Small and Medium Business Support Center, Suwon, Gyeonggi, South Korea. September 25-28.

  
  \end{list2}  


\section{\sc 2016}
\begin{list2}

\item NCERA-225 Meeting. Implementation and Strategies for National Beef Cattle Genetic Evaluation. Stoney Creek Hotel, St. Joseph, MO. October 27-28. 

  \vspace{0.5cm}
  
\item The 5th International Conference on Quantitative Genetics. Monona Terrace Community and Convention Center, Madison, WI. June 12-17. 

\end{list2}  




\section{\sc 2015}
\begin{list2}
  
\item The 29th International Mammalian Genome Conference. Yokohama Port Opening Memorial Hall, Yokohama, Japan. November 8-11. 

  \vspace{0.5cm}
  
  \item DNA Technology: Where we've been, where we are, and where we're headed. The US Meat Animal Research Center, Clay Center, NE. October 19. 
  
  \vspace{0.5cm}
  
\item GO-FAANG Workshop. National Academy of Sciences Building, Washington, DC. October 7-8. 
\end{list2}  

\section{\sc 2014}
\begin{list2}

\item Sheep Genomics Workshop. University of Nebraska-Lincoln. November 13-14.  

\vspace{0.5cm}

\item NCERA-225 Meeting. Implementation and Strategies for National Beef Cattle Genetic Evaluation. Bozeman, MT. October 23-24. 

\end{list2}  




\section{\sc 2009}
\begin{list2}
\item Symposium: ``Statistical Genetics of Livestock for the Post-Genomic Era (SGLPGE)". University of Wisconsin-Madison. May 4-6. 
\end{list2}  

\section{\sc 2008}
\begin{list2}
\item  The 109th Japanese Society of Animal Science Meeting. Tokiwa University, Mito, Ibaraki, Japan. March 27-29. 

\end{list2}










\vspace{0.5cm}
\section{\sc Miscellaneous} 
\begin{list2}
\item Languages: English and Japanese
  \vspace{0.2cm}
\item Computer Skills 
  \begin{itemize}
    \item Programming Languages: Python 
    \item Statistical/Numerical Computational  Tools: R, Octave
    \item Content-description Languages: XML, XHTML, CSS, \LaTeX, Markdown
    \item Operating Systems: Linux and Mac OS X
  \end{itemize}
    \vspace{0.2cm}
\item Courses taken for credits at the University of Wisconsin-Madison 
  \begin{itemize}

\item Spring 2012
\begin{itemize}
\item    Animal Sciences 875-004: Topics in Analysis of Quantitative Genomic Data (Daniel Gianola)
\item    Dairy Science 875-005: Parallel Programming \& High Performance Computing (Xiao-Lin Nick Wu)
\end{itemize}


\item Fall 2011
\begin{itemize}
\item    Dairy Science 875-005: Molecular Aspects of Animal Breeding (Hasan Khatib)
\item    Statistics 840: Statistical Model Building and Learning (Grace Wahba)
\end{itemize}



\item Spring 2011
\begin{itemize}
\item    Mathematics 609: Mathematical Methods in Systems Biology (Gheorghe Craciun)
\item    Statistics 610: Introduction to Statistical Inference (Chunming Zhang)
\item    Statistics 992-001: Statistical Methods for QTL Mapping (Karl Broman)
\end{itemize}


\item Fall 2010
\begin{itemize}
\item    Statistics 609: Mathematical Statistics I (Chunming Zhang)
\item    Statistics 701: Applied Time Series Analysis, Forecasting \& Control I (Yazhen Wang)
\item    Statistics 775: Introduction to Bayesian Decision \& Control (Kam-Wah Tsui)
\end{itemize}


\item Summer 2010
\begin{itemize}
\item    Population Health Sciences 904: Analytic Methods in Genetic Epidemiology (Corinne Engelman, Karl Broman, Bret Payseur, Kristin Meyers)
\end{itemize}


\item Spring 2010
\begin{itemize}

\item    Animal Sciences 875: Linear Models with Applications in Biology and Agriculture (Daniel Gianola)
\item    Statistics 850: Theory \& Application of Regression and Analysis of Variance II (Wei-Yin Loh)

\end{itemize}


\item Fall 2009
\begin{itemize}
\item    Computer Science 576: Introduction to Bioinformatics (Colin Dewey)
\item    Dairy Science 875-006: Design \& Analysis of Microarray Experiments in Agriculture (Guilherme J. M. Rosa)
\item    Dairy Science 875-011: Introduction to Bayesian Data Analysis with R (Xiao-Lin Nick Wu)
\item    Genetics 629: Evolutionary Genetics (John Doebley, Bret Larget, Bret Payseur)
\item    Statistics 849: Theory \& Application of Regression and Analysis of Variance I (Sunduz Keles)
\end{itemize}




\item Summer 2009
\begin{itemize}
\item Computer Science 367: Introduction to Data Structure
\end{itemize}

\item Spring 2009
\begin{itemize}
\item Agronomy 771: Experimental Design (Mike Casler)
\item Agronomy 772: Applications in ANOVA (Mike Casler)
\item Mathematics 222: Calculus and Analytic Geometry
\item Statistics 771: Statistical Computing (Michael Newton)
\end{itemize}


\item Fall 2008
\begin{itemize}
\item Statistics 424: Statistical Experimental Design for Engineers (Peter Z. G. Qian)
\item Statistics 541: Introduction to Biostatistics (Ismor Fischer)
\item Zoology 645: Modeling in Population Genetics \& Evolution (Andrew Peters)
\end{itemize}

\item Summer 2008
\begin{itemize}
\item Computer Science 302: Introduction to Programming
\item Mathematics 431: Introduction to the Theory of Probability
\end{itemize}

\end{itemize}
\end{list2}



\vspace{0.5cm}
\section{ \sc Additional training}
\vspace{1cm}

\section{\sc 2019}
\begin{list2}

\item Special Lecture on Quantitative and Statisitical Genetics. The University of Tokyo, Bunkyo-ku, Tokyo, Japan. October 17-18. Taught by Daniel Gianola.

  \vspace{0.5cm}
  
\item Phenome Digital Phenotyping Workshop. Phenome 2019. El Conquistador Tucson, A Hilton Resort, Tucson, AZ. February 6. Taught by Malia Gehan, Noah Fahlgren, Joshua Peschel, Sierra Young, Magdalena Julkowska and Alina Zare.
\end{list2}  


\section{\sc 2016}
\begin{list2}
\item Next Generation Plant and Animal Breeding Programs Workshop. University of Nebraska-Lincoln. March 21-25.  Taught by John Hickey, Gregor Gorjanc, and Chris Gaynor.
\end{list2}  


\section{\sc 2015}
\begin{list2}
\item Participant of the Research Development Fellows Program (RDFP) 
\end{list2}

\section{\sc 2014}
\begin{list2}

\item Participant of Fall 2014 Adopting Research Based Instructional Strategies for Enhancing (ARISE) Professional Development Programs - Just in Time Teaching (JiTT)

\vspace{0.5cm}

\item 19th Summer Institute in Statistical Genetics: 
``Module 23: Advanced Quantitative Genetics ". University of Washington.  July 23-25.
Taught by Mike Goddard and Peter Visscher. 

\vspace{0.5cm}

\item 19th Summer Institute in Statistical Genetics: 
``Module 19: Statistical \& Quantitative Genetics of Disease". University of Washington. July 21-23.
Taught by John Witte and Naomi Wray. 

\vspace{0.5cm}

\item UC Davis Bioinformatics Training Program: 
``Using Galaxy for Analysis of High Throughput Sequence Data". University of California, Davis. June 16-20. 
Taught by the Bioinformatics Core. 

\vspace{0.5cm}

\item Short course: ``Evolutionary Quantitative Genetics". University of Wisconsin-Madison. May 19-23. 
Taught by Bruce Walsh. 

\end{list2}

\section{\sc 2013}
\begin{list2}
\item Short course: ``Statistical methods for prediction of complex traits using whole-genome molecular markers". University of Wisconsin-Madison. May 27-31. 
Taught by Daniel Gianola and Gustavo de los Campos.
\end{list2}  

\section{\sc 2012}
\begin{list2}
\item Short course: ``Introduction to genome-enabled selection \& Inferring causal phenotype networks using structural equation models". 
Kyoto University. August 31. Taught by Guilherme J.M. Rosa. 

\vspace{0.5cm}

\item Short course: ``Identifying Genes for Complex and Mendelian Traits Using Next Generation Sequence Data". 26th International Biometric Conference. Kobe International Conference Center, Kobe Japan. August 26. Taught by Suzanne Leal.  

\vspace{0.5cm}

\item Short course: ``Programming and computer algorithms with focus on genomic selection in animal breeding".  University of Georgia. May 15 - June 1.  
Taught by Ignacy Misztal, Shogo Tsuruta, Ignacio Aguilar, Zulma Vitezica, and  Andres Legarra. 
\end{list2}  

\section{\sc 2006}
\begin{list2}
\item Short course: ``Estimation of Variance Conponents in Animal Breeding". Obihiro University of Agriculture and Veterinary Medicine. November. 
Taught by Shogo Tsuruta.  
\end{list2}  




\vspace{0.5cm}
\section{\sc References}
References and additional information available upon request. 

\begin{comment}
\begin{tabular}{ll}
% Sewall Wright Professor of Animal Breeding and Genetics
[1] & {\bf \underline{Daniel Gianola}}, Ph.D. Sewall Wright Professor of Animal Breeding and Genetics. \\
    & Department of Animal Sciences, University of Wisconsin-Madison. \\ 
    & Address: 1675 Observatory Dr. Madison, WI 53706-1284.  \\
    & E-mail: gianola -at- ansci.wisc.edu, Phone: 608-265-2054
\end{tabular}
\end{comment}



\end{resume}
\end{document}




