<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html lang="en">
<head>
<link rel="stylesheet" href="../stylesheets/styles.css">
<link rel="stylesheet" href="../stylesheets/pygment_trac.css">
<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
   
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="Content-Style-Type" content="text/css">
<title>Gota Morota - List of Publications by Year</title>
</head>

<body>
<p class="menublock"><a href="../index.html">Back</a></p>




<h1>Publications</h1>
<ul>
  <li><a href="http://scholar.google.com/citations?user=isy-n24AAAAJ&hl=en">Google Scholar</a></li>
  <li><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Morota+G%5BAuthor%5D&sort=date">PubMed</a></li>
  <li><a href="http://europepmc.org/search/?sortby=Date&page=1&query=AUTH:%22Morota+G%22">Europe PubMed Central</a></li>
  <li><a href="https://researchmap.jp/gmorota">researchmap</a></li>
  <li><a href="https://www.semanticscholar.org/author/G.-Morota/2166592">Semantic Scholar
  <li><a href="https://www.scopus.com/authid/detail.uri?authorId=55356558800">Scopus</a></li>
</a></li>
  <li><a href="https://publons.com/researcher/2656306/gota-morota/">Publons</a></li>
  <li><a href="http://orcid.org/0000-0002-3567-6911">ORCID</a></li>
  <li><a href="https://www.rxivist.org/authors/247419">Rxivist</a></li>
  <li><a href="http://www.prepubmed.org/search_results/?q=Morota+G">PrePubMed</a></li>
  <li><a href="journals.html">Journal list</a></li>
  <li><a href="apc.html">Journal APC</a></li>
 </ul>


<h2>Preprints</h2>
<p>
  <ul>


   <li>
       Murphy MD, Fernandes SB, <strong><u>Morota G</u></strong>, and Lipka AE. Simulation study evaluating the ability of two statistical approaches to identify variance quantitative trait loci Arabidopsis and maize. bioRxiv.
      [<a href="https://doi.org/10.1101/2021.06.25.449982">bioRxiv</a>]
   </li>


    <li>
      Sabag I, <strong><u>Morota G</u></strong>, and Peleg Z. Genome-wide association uncovers the genetic architecture of tradeoff between flowering date and yield components in sesame. bioRxiv.
      [<a href="https://doi.org/10.1101/2021.04.22.440889">bioRxiv</a>]
    </li>  
    
    <li>
      <strong><u>Morota G</u></strong>, Jarquin D, Campbell MT, and Iwata H. Statistical methods for the quantitative genetic analysis of high-throughput phenotyping data. arXiv.
      [<a href="https://arxiv.org/abs/1904.12341">arXiv</a>]
    </li>   
  
  </ul>
</p>



<h2>2021</h2>
<p>
  <ul>

    <li> Mota LFM, Pegolo S, Baba T, <strong><u>Morota G</u></strong>, Penagaricano F, Bittante G, and Cecchinato A. 2021. Comparison of single-breed and multi-breed training population for infrared predictions of novel phenotypes in Holstein cows. Animals. 11:1993. 
      [<a href="https://doi.org/10.3390/ani11071993">DOI</a>]
    </li>

    

    <li> Silva FF, <strong><u>Morota G</u></strong>, and Rosa GJM. 2021. Editorial: High-throughput phenotyping in the genomic improvement of livestock. Frontiers in Genetics. 12:707343.
      [<a href="https://doi.org/10.3389/fgene.2021.707343">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/34220969/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/34220969/">EuropePMC</a>]
    </li>

  

    <li> Mota LFM, Pegolo S, Baba T, Penagaricano F, <strong><u>Morota G</u></strong>, Bittante G, and Cecchinato A. 2021. Evaluating the performance of machine learning and variable selection methods for predicting difficult-to-measure traits in Holstein dairy cattle using milk infrared spectral data. Journal of Dairy Science. 104:8107-8121.
      [<a href="https://doi.org/10.3168/jds.2020-19861">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/33865589/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/33865589/">EuropePMC</a>]
    </li>


    <li> Baba T, Pegolo S, Mota LFM, Penagaricano F, Bittante G, Cecchinato A, and <strong><u>Morota G</u></strong>. 2021. Integrating genomic and infrared spectral data improves the prediction of milk protein composition in dairy cattle. Genetics Selection Evolution. 53:29. 
      [<a href="https://doi.org/10.1186/s12711-021-00620-7">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/33726672/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/33726672/">EuropePMC</a>]
  </li>  

    <li> <strong><u>Morota G</u></strong>, Cheng H, Cook D, and Tanaka E. 2021. ASAS-NANP SYMPOSIUM: Prospects for interactive and dynamic graphics in the era of data-rich animal science. Journal of Animal Science. 99:1-17.
      [<a href="https://doi.org/10.1093/jas/skaa402">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/33626150/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/33626150/">EuropePMC</a>]
    </li>   

     <li>
      Goncalves MTV, <strong><u>Morota G</u></strong>, Almeida Costa PM, Vidigal PMP, Barbosa MHP, and Peternelli LA. 2021. Near-infrared spectroscopy outperforms genomics for predicting sugarcane feedstock quality traits. PLOS ONE. 16(3):e0236853.
      [<a href="https://doi.org/10.1101/2020.07.16.206110">bioRxiv</a>]
      [<a href="https://doi.org/10.1371/journal.pone.0236853">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/33661948/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/33661948/">EuropePMC</a>]
    </li>

     
    <li>
      Yu H and <strong><u>Morota G</u></strong>. 2021. GCA: An R package for genetic connectedness analysis using pedigree and genomic data. BMC Genomics. 22:119.
      [<a href="https://doi.org/10.1186/s12864-021-07414-7">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/33588757/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/33588757/">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/696419">bioRxiv</a>]
    </li>

    <li>
      Yu H, Lee K, and <strong><u>Morota G</u></strong>. 2021. Forecasting dynamic body weight of non-restrained pigs from images using an RGB-D sensor camera. Translational Animal Science. 5:1-9.
      [<a href="https://doi.org/10.1093/tas/txab006">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/33659861/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/33659861/">EuropePMC</a>]
    </li>
 
    <li>
      Pegolo S, Yu H, <strong><u>Morota G</u></strong>, Bisutti V, Rosa GJM, Bittante G, and Cecchinato A. 2021. Structural equation modelling for unravelling the multivariate genomic architecture of milk proteins in dairy cattle. Journal of Dairy Science. 104:5705-5718.
      [<a href="https://doi.org/10.3168/jds.2020-18321">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/33663837/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/33663837/">EuropePMC</a>]
    </li>

     <li>
       Zhu F, Paul P, Hussain W, Wallman K, Dhatt BK, Sandhu J, Irvin L, <strong><u>Morota G</u></strong>, Yu H, and Walia H. 2021. SeedExtractor: an open-source GUI for seed image analysis. Frontiers in Plant Science. 11:581546.
       [<a href="https://doi.org/10.3389/fpls.2020.581546">DOI</a>]
       [<a href="https://pubmed.ncbi.nlm.nih.gov/33597957/">PubMed</a>]
       [<a href="http://europepmc.org/abstract/MED/33597957/">EuropePMC</a>]
       [<a href="https://doi.org/10.1101/2020.06.28.176230">bioRxiv</a>]
     </li>
     

     <li>
       Momen M, Bhatta M, Hussain W, Yu H, and <strong><u>Morota G</u></strong>. Modeling multiple phenotypes in wheat using data-driven genomic exploratory factor analysis and Bayesian network learning. 2021. Plant Direct. 00:e00304.
       [<a href="https://doi.org/10.1002/pld3.304">DOI</a>]
       [<a href="https://pubmed.ncbi.nlm.nih.gov/33532691/">PubMed</a>]
       [<a href="http://europepmc.org/abstract/MED/33532691/">EuropePMC</a>]
       [<a href="https://doi.org/10.1101/2020.09.03.282335">bioRxiv</a>]
    </li>



     <li>
      Dhatt B, Paul P, Sandhu J, Hussain W, Irvin L, Zhu F, Adviento-Borbe M, Lorence A, Staswick P, Yu H, <strong><u>Morota G</u></strong>, and Walia H. 2021. Allelic variation in rice <i>Fertilization independent endosperm 1</i> contributes to grain width under high night temperature stress. New Phytologist. 229:335-350. 
      [<a href="https://doi.org/10.1111/nph.16897">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/32858766/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/32858766">EuropePMC</a>]
    </li>

     
  </ul>
</p>

    

<h2>2020</h2>
<p>
  <ul>

    <li>
      Wang Z, Chapman D, <strong><u>Morota G</u></strong>, and Cheng H. 2020. A Multiple-trait Bayesian variable selection regression method for integrating phenotypic causal networks in genome-wide association studies. G3: Genes, Genomes, Genetics. 10:4439-4448. 
      [<a href="https://doi.org/10.1534/g3.120.401618">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/33020191/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/33020191">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/847285">bioRxiv</a>]
    </li>

      
    <li>
      Amorim ST, Yu H, Momen M, de Albuquerque, LG, Pereira, ASC, Baldi F, and <strong><u>Morota G</u></strong>. 2020. An assessment of genomic connectedness measures in Nellore cattle. Journal of Animal Science. 98:1-12.
      [<a href="https://doi.org/10.1093/jas/skaa289">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/32877515/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/32877515">EuropePMC</a>]
    </li>
    
   
    
    <li>
      Campbell MT, Grondin A, Walia H, and <strong><u>Morota G</u></strong>. 2020. Leveraging genome-enabled growth models to study shoot growth responses to water deficit in rice (Oryza sativa). Journal of Experimental Botany. 71:5669-5679.
      [<a href="https://doi.org/10.1093/jxb/eraa280">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/32526013/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/32526013">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/690479">bioRxiv</a>]
    </li>
    
    <li>
      Yu H, <strong><u>Morota G</u></strong>, Celestino Jr. EF, Dahlen CR, Wagner SA, Riley DG, and Hulsman Hanna LL. 2020. Deciphering cattle temperament measures derived from a four-platform standing scale using genetic factor analytic modeling. Frontiers in Genetics. 11:599.
      [<a href="https://doi.org/10.3389/fgene.2020.00599">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/32595702/">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/32595702">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/2020.01.20.913343">bioRxiv</a>]
    </li>
      
    
      <li>
	Pegolo S, Momen M, <strong><u>Morota G</u></strong>, Rosa GJM, Gianola G, Bittante G, and Cecchinato A. 2020. Structural equation modeling for investigating multi-trait genetic architecture of udder health in dairy cattle. Scientific Reports. 10:7751.
	[<a href="https://doi.org/10.1038/s41598-020-64575-3">DOI</a>]
	[<a href="https://pubmed.ncbi.nlm.nih.gov/32385377/">PubMed</a>]
	[<a href="http://europepmc.org/abstract/MED/32385377">EuropePMC</a>]
      </li>
      

      <li>
	Roudbar MA, Mohammadabadi MR, Mehrgardi AA, Abdollahi-Arpanahi R, Momen M, <strong><u>Morota G</u></strong>, Lopes FB, Gianola D, and Rosa GJM. 2020. Integration of single nucleotide variants and whole-genome DNA methylation profiles for classification of rheumatoid arthritis cases from controls. Heredity. 124:658-674.
	[<a href="https://doi.org/10.1038/s41437-020-0301-4">DOI</a>]
	[<a href="https://pubmed.ncbi.nlm.nih.gov/32127659/">PubMed</a>]
       [<a href="http://europepmc.org/abstract/MED/32127659">EuropePMC</a>]
      </li>
   
    
     <li>
       Hussain W, Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. 2020. Variance heterogeneity genome-wide mapping for cadmium in bread wheat reveals novel genomic loci and epistatic interactions. The Plant Genome. 13:e20011.
       [<a href="https://doi.org/10.1002/tpg2.20011">DOI</a>]
       [<a href="https://doi.org/10.1101/668087">bioRxiv</a>]
     </li>

     
     <li>
       Baba T, Momen M, Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. 2020. Multi-trait random regression models increase genomic prediction accuracy for a temporal physiological trait derived from high-throughput phenotyping. PLOS ONE. 15(2):e0228118.
       [<a href="https://doi.org/10.1371/journal.pone.0228118">DOI</a>]
       [<a href="https://pubmed.ncbi.nlm.nih.gov/32012182/">PubMed</a>]
       [<a href="http://europepmc.org/abstract/MED/32012182">EuropePMC</a>]
       [<a href="https://doi.org/10.1101/772038">bioRxiv</a>]
    </li>

     
    <li>
     Paul P, Dhatt B, Sandhu J, Hussain W, Irvin L, <strong><u>Morota G</u></strong>, Staswick P, and Walia H. 2020. Divergent phenotypic response of rice accessions to transient heat stress during early seed development. Plant Direct. 4:1–13.
     [<a href="https://doi.org/10.1002/pld3.196">DOI</a>]
     [<a href="https://pubmed.ncbi.nlm.nih.gov/31956854">PubMed</a>]
     [<a href="http://europepmc.org/abstract/MED/31956854">EuropePMC</a>]
    </li>
  </ul>
</p>




<h2>2019</h2>
<p>
  <ul>

    <li>
      Atagi Y, <strong><u>Morota G</u></strong>, Onogi A, Osawa T, Yasumori T, Adachi K, Yamaguchi S, Aihara M, Goto H, Togashi K, and Iwata H. 2019. Consideration of heat stress in multiple lactation test–day models for dairy production traits. Interbull Bulletin. 55:81-87.
      [<a href="https://journal.interbull.org/index.php/ib/article/view/1467">PDF</a>]
    </li>

    
    <li>
      Momen M, Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. 2019. Utilizing trait networks and structural equation models as tools to interpret multi-trait genome-wide association studies. Plant Methods. 15:107.
      [<a href="https://doi.org/10.1186/s13007-019-0493-x">DOI</a>]
      [<a href="https://pubmed.ncbi.nlm.nih.gov/31548847">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/31548847">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/553008">bioRxiv</a>]
    </li>
   
      
    <li>
      Momen M, Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. 2019. Predicting longitudinal traits derived from high-throughput phenomics in contrasting environments using genomic Legendre polynomials and B-splines. G3: Genes, Genomes, Genetics. 9:3369-3380.
      [<a href="https://doi.org/10.1534/g3.119.400346">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/31427454">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/31427454">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/632117">bioRxiv</a>]
    </li>

    
    <li>
      Yu H, Campbell MT, Zhang Q, Walia H, and <strong><u>Morota G</u></strong>. 2019. Genomic Bayesian confirmatory factor analysis and Bayesian network to characterize a wide spectrum of rice phenotypes. G3: Genes, Genomes, Genetics. 9:1975-1986.
      [<a href="https://doi.org/10.1534/g3.119.400154">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/30992319">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/30992319">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/435792">bioRxiv</a>]
    </li>

    
    <li>
      Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. 2019. Leveraging breeding values obtained from random regression models for genetic inference of longitudinal traits. The Plant Genome. 12:180075. 
      [<a href="https://doi.org/10.3835/plantgenome2018.10.0075">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/31290928">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/31290928">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/435685">bioRxiv</a>]
    </li>

 </ul>
</p>



<h2>2018</h2>
<p>
  <ul>

    <li>
      Hussain W, Campbell MT, Walia H, and <strong><u>Morota G</u></strong>. 2018. ShinyAIM: Shiny-based application of interactive Manhattan plots for longitudinal genome-wide association studies. Plant Direct. 2:1–4.
      [<a href="https://doi.org/10.1002/pld3.91">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/31245691">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/31245691">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/383026">bioRxiv</a>]
    </li>

    
    <li>
      Momen M, Mehrgardi AA, Roudbar MA, Kranis A, Pinto RM, Valente BD, <strong><u>Morota G</u></strong>, Rosa GJM, and Gianola D. 2018. Including phenotypic causal networks in genome-wide association studies using mixed effects structural equation models. Frontiers in Genetics. 9:455.
      [<a href="https://doi.org/10.3389/fgene.2018.00455">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/30356716">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/30356716">EuropePMC</a>]
      [<a href="https://doi.org/10.1101/251421">bioRxiv</a>]
    </li>   
    
      
    <li> Momen M and <strong><u>Morota G</u></strong>. 2018. Quantifying genomic connectedness and prediction accuracy from additive and non-additive gene actions. Genetics Selection Evolution. 50:45. 
      [<a href="https://doi.org/10.1186/s12711-018-0415-9">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/30223766">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/30223766">EuropePMC</a>]
   </li>
   

    <li>Yu H, Spangler ML, Lewis RM, and <strong><u>Morota G</u></strong>. 2018. Do stronger measures of genomic connectedness enhance prediction accuracies across management units? Journal of Animal Science. 96:4490-4500.
      [<a href="https://doi.org/10.1093/jas/sky316">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/30165381">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/30165381">EuropePMC</a>]
  </li>


   <li> Momen M, Mehrgardi AA, Sheikhy ASA, Kranis A, Tusell L,  <strong><u>Morota G</u></strong>, Rosa GJM, and Gianola D. 2018. Predictive ability of genome-assisted statistical models under various forms of gene action. Scientific Reports. 8:12309. 
     [<a href="https://doi.org/10.1038/s41598-018-30089-2">DOI</a>]
     [<a href="https://www.ncbi.nlm.nih.gov/pubmed/30120288">PubMed</a>]
     [<a href="http://europepmc.org/abstract/MED/30120288">EuropePMC</a>]
  </li>


   <li> Campbell MT, Walia H, and  <strong><u>Morota G</u></strong>. 2018. Utilizing random regression models for genomic prediction of a longitudinal trait derived from high-throughput phenotyping. Plant Direct. 2:1-11.
     [<a href="https://doi.org/10.1002/pld3.80">DOI</a>]
     [<a href="https://www.ncbi.nlm.nih.gov/pubmed/31245746">PubMed</a>]
     [<a href="http://europepmc.org/abstract/MED/31245746">EuropePMC</a>]
     [<a href="https://doi.org/10.1101/319897">bioRxiv</a>]
    </li>
    

    <li> Alvarenga AB, Rovadoscki GA, Petrini J, Coutinho LL, <strong><u>Morota G</u></strong>, Spangler ML, Pinto LFB, Carvalho GGP, and Mourao GB. 2018. Linkage disequilibrium in Brazilian Santa Ines breed, Ovis aries. Scientific Reports. 8:8851.
      [<a href="https://doi.org/10.1038/s41598-018-27259-7">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29892085">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/29892085">EuropePMC</a>]
    </li>

    
    <li>Rovadoscki GA, Pertille SFN, Alvarenga AB,  Cesar ASM, Pertille F, Petrini J, Franzo V, Soares WVB, <strong><u>Morota G</u></strong>, Spangler ML, Pinto LFB, de Carvalho GGP, Lanna DPD, Coutinho LL, and Mourao GB. 2018. Estimates of genomic heritability and genome-wide association study for fatty acids profile in Santa Ines sheep. BMC Genomics. 19:375.
      [<a href="https://doi.org/10.1186/s12864-018-4777-8">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29783944">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/29783944">EuropePMC</a>]
    </li>

    <li>Yu H, Spangler ML, Lewis RM, and <strong><u>Morota G</u></strong>. 2018. Stronger measures of genomic connectedness enhance prediction accuracies across management units. Proceedings, 11th World Congress of Genetics Applied to Livestock Production. 11:406. Auckland, New Zealand. February 11-16. 
       [<a href="http://www.wcgalp.org/proceedings/2018/stronger-measures-genomic-connectedness-enhance-prediction-accuracies-across">WCGALP</a>] 
       [<a href="./pdf/yu2018WCGALP.pdf">PDF</a>] 
    </li>
    
    <li>Abdollahi-Arpanahi R, <strong><u>Morota G</u></strong>, and  Penagaricano F. 2018. Predicting bull fertility using biologically informed genomic models.  Proceedings, 11th World Congress of Genetics Applied to Livestock Production. 11:683. Auckland, New Zealand. February 11-16. 
      [<a href="http://www.wcgalp.org/proceedings/2018/predicting-bull-fertility-using-biologically-informed-genomic-models">WCGALP</a>] 
    </li>

     <li>Mamani GC, Santana BF, Oliveira Junior GA, Mattos E, Ventura RV, Eler JP, <strong><u>Morota G</u></strong>, and Ferraz JBS. 2018. Effect of inbreeding in productive traits in Nellore cattle.  Proceedings, 11th World Congress of Genetics Applied to Livestock Production. 11:855. Auckland, New Zealand. February 11-16. 
       [<a href="http://www.wcgalp.org/proceedings/2018/effect-inbreeding-productive-traits-nellore-cattle">WCGALP</a>] 
     </li>

     
    <li><strong><u>Morota G</u></strong>, Ventura RV, Silva FF, Koyama M, and Fernando SC. 2018. BIG DATA ANALYTICS AND PRECISION ANIMAL AGRICULTURE SYMPOSIUM: Machine learning and data mining advance predictive big data analysis in precision animal agriculture. Journal of Animal Science. 96:1540–1550.
      [<a href="http://dx.doi.org/10.1093/jas/sky014">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29385611">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/29385611">EuropePMC</a>]
    </li>

    <li>He J, Xu J, Wu XL, Bauck S, Lee J, <strong><u>Morota G</u></strong>, Kachman SD, and Spangler ML. 2018. Comparing strategies for selection of low-density SNPs for imputation-mediated genomic prediction in U.S. Holsteins. Genetica. 146:137-149.
      [<a href="https://dx.doi.org/10.1007/s10709-017-0004-9">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29243001">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/29243001">EuropePMC</a>]
    </li>   
  
</ul>
</p>



<h2>2017</h2>
<p>
  <ul>
    
    <li><strong><u>Morota G</u></strong>. ShinyGPAS: 2017. Interactive genomic prediction accuracy simulator based on deterministic formulas. Genetics Selection Evolution. 49:91. 
      [<a href="https://doi.org/10.1186/s12711-017-0368-4">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29262775">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/29262775">EuropePMC</a>]
      [<a href="https://dx.doi.org/10.1101/164772">bioRxiv</a>]
    </li>
    
    
    <li>
      Abdollahi-Arpanahi R, <strong><u>Morota G</u></strong>, and Penagaricano F. 2017. Predicting bull fertility using genomic data and biological information. Journal of Dairy Science. 100:9656-9666. 
      [<a href="https://doi.org/10.3168/jds.2017-13288">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28987577">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/28987577">EuropePMC</a>]
    </li>


    
    <li>Yu H, Spangler ML, Lewis RM, and <strong><u>Morota G</u></strong>. 2017. Genomic relatedness strengthens genetic connectedness across management units.  G3: Genes, Genomes, Genetics. 10:3543-3556.
      [<a href="https://doi.org/10.1534/g3.117.300151">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28860185">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/28860185">EuropePMC</a>]
      [<a href="https://dx.doi.org/10.1101/130138">bioRxiv</a>]
    </li>



    
    <li>Beissinger TM and <strong><u>Morota G</u></strong>. 2017. Medical subject heading (MeSH) annotations illuminate maize genetics and evolution. Plant Methods. 13:8. 
      [<a href="http://dx.doi.org/10.1186/s13007-017-0159-5">DOI</a>]
      [<a href="https://www.ncbi.nlm.nih.gov/pubmed/28250803">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/28250803">EuropePMC</a>]
      [<a href="http://dx.doi.org/10.1101/048132">bioRxiv</a>]
    </li>


  </ul>
</p>



 <h2>2016</h2>
<p>
  <ul>

    <li><strong><u>Morota G</u></strong>, Beissinger TM, and Penagaricano F. 2016.  MeSH-informed enrichment analysis and MeSH-guided semantic similarity among functional terms and gene products in chicken. G3: Genes, Genomes, Genetics. 6:2447-2453. 
      [<a href="http://dx.doi.org/10.1534/g3.116.031096">DOI</a>]
      [<a href="http://www.ncbi.nlm.nih.gov/pubmed/27261003">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/27261003">EuropePMC</a>]
      [<a href="http://dx.doi.org/10.1101/034975">bioRxiv</a>]
    </li>

    <li>Abdollahi-Arpanahi R, <strong><u>Morota G</u></strong>, Valente BD, Kranis A, Rosa GJM, and Gianola D. 2016. Differential contribution of genomic regions to marked genetic variation and prediction of quantitative traits in broiler chickens. Genetics Selection Evolution. 48:10. 
      [<a href="http://dx.doi.org/10.1186/s12711-016-0187-z">DOI</a>]
      [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26842494">PubMed</a>]
      [<a href="http://europepmc.org/abstract/MED/26842494">EuropePMC</a>]
    </li>


  </ul>
</p>


  <h2>2015</h2>
<p>
<ul>

  <li>Hu Y, <strong><u>Morota G</u></strong>, Rosa GJM, and Gianola D. 2015. Prediction of plant height in Arabidopsis thaliana from DNA methylation data. Genetics. 201:779-793.
    [<a href="http://dx.doi.org/10.1534/genetics.115.177204">DOI</a>]
    [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26253546">PubMed</a>]
    [<a href="http://europepmc.org/abstract/MED/26253546">EuropePMC</a>]
  </li>

  <li>Valente BD, <strong><u>Morota G</u></strong>, Penagaricano F, Gianola D, Weigel KA, and Rosa GJM. 2015. The causal meaning of genomic predictors and how it affects  construction and comparison of genome-enabled selection models. Genetics. 200:483-494. 
    [<a href="http://dx.doi.org/10.1534/genetics.114.169490">DOI</a>]
    [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25908318">PubMed</a>]
    [<a href="http://europepmc.org/abstract/MED/25908318">EuropePMC</a>]
    [<a href="http://dx.doi.org/10.1101/001511">bioRxiv</a>] 
    [<a href="http://arxiv.org/abs/1401.1165">arXiv</a>] 
  </li>


  <li><strong><u>Morota G</u></strong>, Penagaricano F, Petersen JL, Ciobanu DC, 2015. Tsuyuzaki K, and Nikaido I. An application of MeSH enrichment analysis in livestock. Animal Genetics. 46:381-387.
    [<a href="http://dx.doi.org/10.1111/age.12307">DOI</a>]
    [<a href="http://www.ncbi.nlm.nih.gov/pubmed/26036323">PubMed</a>]
    [<a href="http://europepmc.org/abstract/MED/26036323">EuropePMC</a>]
</li>


<li>Abdollahi-Arpanahi R, <strong><u>Morota G</u></strong>, Valente BD, Kranis A, Rosa GJM, and Gianola D. 2015. Assessment of bagging GBLUP for whole-genome prediction of broiler chicken traits. Journal of Animal Breeding and Genetics. 132:218-228.
  [<a href="http://dx.doi.org/10.1111/jbg.12131">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25727456">PubMed</a>]
  [<a href="http://europepmc.org/abstract/MED/25727456">EuropePMC</a>]
</li>

<li>Tsuyuzaki K, <strong><u>Morota G</u></strong>, Ishii M, Nakazato T, Miyazaki S, and Nikaido I. 
  2015. MeSH ORA framework: R/Bioconductor packages to support MeSH over-representation analysis.
  BMC Bioinformatics. 16:45. 
  [<a href="http://dx.doi.org/10.1186/s12859-015-0453-z">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25887539">PubMed</a>]
  [<a href="http://europepmc.org/abstract/PMC/PMC4343279">EuropePMC</a>]
</li>

</ul>
</p>




<h2>2014</h2>
<p>
<ul>

<li><strong><u>Morota G</u></strong> and Gianola D. 2014.  
Kernel-based whole-genome prediction of complex traits: a review. Frontiers in Genetics. 5:363. 
  [<a href="http://dx.doi.org/10.3389/fgene.2014.00363">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/25360145">PubMed</a>] 
  [<a href="http://europepmc.org/abstract/MED/25360145">EuropePMC</a>]
</li>


<li>Gianola D,  <strong><u>Morota G</u></strong>, and Crossa J. 2014. 
Genome-enabled Prediction of Complex Traits with Kernel Methods: What Have We Learned? 
  Proceedings, 10th World Congress of Genetics Applied to Livestock Production. Vancouver, BC, Canada. August 17-22.
  [<a href="http://www.wcgalp.org/proceedings/2014/genome-enabled-prediction-complex-traits-kernel-methods-what-have-we-learned">WCGALP</a>] 
  [<a href="./pdf/gianola2014WCGALP.pdf">PDF</a>]
</li>

<li>Valente BD,  <strong><u>Morota G</u></strong>, Rosa GJM, Gianola D, and  Weigel KA. 2014. 
Causal meaning of genomic predictors: Implication on genome-enabled selection modeling. 
  Proceedings, 10th World Congress of Genetics Applied to Livestock Production. Vancouver, BC, Canada. August 17-22. 
  [<a href="http://www.wcgalp.org/proceedings/2014/causal-meaning-genomic-predictors-implication-genome-enabled-selection-modeling">WCGALP</a>] 
  [<a href="./pdf/valente2014WCGALP.pdf">PDF</a>] 
</li>

<li><strong><u>Morota G</u></strong>, Boddhireddy P, Vukasinovic N, Gianola D, and DeNise S. 2014. 
Kernel-based variance components estimation and whole-genome prediction of pre-corrected 
phenotypes and progeny tests for dairy cow health traits. Frontiers in Genetics. 5:56. 
  [<a href="http://dx.doi.org/10.3389/fgene.2014.00056">DOI</a>] 
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24715901">PubMed</a>] 
  [<a href="http://europepmc.org/abstract/MED/24715901">EuropePMC</a>]
</li>

<li><strong><u>Morota G</u></strong>, Abdollahi-Arpanahi R, Kranis A, and Gianola D. 2014.  
Genome-enabled prediction of broiler traits in chickens using genomic annotation.  BMC Genomics. 15:109. 
  [<a href="http://dx.doi.org/10.1186/1471-2164-15-109">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/24502227">PubMed</a>] 
  [<a href="http://europepmc.org/abstract/MED/24502227">EuropePMC</a>]
</li>

<li>Abdollahi-Arpanahi R, Pakdel A, Nejati-Javaremi A, Moradi-Shahrbabak M, 
<strong><u>Morota G</u></strong>, Valente BD, Kranis A, Rosa GJM, and Gianola D. 2014. 
Dissection of additive genetic variability for quantitative traits in chickens using SNP markers. 
Journal of Animal Breeding and Genetics. 131:183-193. 
[<a href="http://dx.doi.org/10.1111/jbg.12079">DOI</a>]
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/24460953">PubMed</a>] 
[<a href="http://europepmc.org/abstract/MED/24460953">EuropePMC</a>]
</li>

<li>Abdollahi-Arpanahi R, Nejati-Javaremi A, Pakdel A, Moradi-Shahrbabak M, 
<strong><u>Morota G</u></strong>, Valente BD, Kranis A, Rosa GJM, and Gianola D. 2014. 
Effect of allele frequencies, effect sizes and number of markers on prediction of quantitative traits in chickens. 
Journal of Animal Breeding and Genetics. 131:123-133. 
[<a href="http://dx.doi.org/10.1111/jbg.12075">DOI</a>]
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/24397350">PubMed</a>] 
[<a href="http://europepmc.org/abstract/MED/24397350">EuropePMC</a>]
</li>

</ul>
</p>





<h2>2013</h2>
<p>
<ul>

<li><strong><u>Morota G</u></strong>, Koyama M, Rosa GJM,  Weigel KA, and Gianola D. 2013. 
Predicting complex traits using a diffusion kernel on genetic markers with an application to dairy cattle and wheat data. 
Genetics Selection Evolution. 45:17.  
  [<a href="http://dx.doi.org/10.1186/1297-9686-45-17">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23763755">PubMed</a>] 
  [<a href="http://europepmc.org/abstract/MED/23763755">EuropePMC</a>]
</li>


<li><strong><u>Morota G</u></strong> and Gianola D. 2013. 
Evaluation of linkage disequilibrium in wheat with an L1 regularized sparse Markov network. Theoretical and Applied Genetics. 126:1991-2002. 
  [<a href="http://dx.doi.org/10.1007/s00122-013-2112-y">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23661079">PubMed</a>] 
  [<a href="http://europepmc.org/abstract/MED/23661079">EuropePMC</a>]
</li>

</ul>
</p>


<h2>2012</h2>
<p>
<ul>

<li><strong><u>Morota G</u></strong>, Valente BD, Rosa GJM,  Weigel KA, and Gianola D. 2012. 
 An assessment of linkage disequilibrium in Holstein cattle using a Bayesian network. 
  Journal of Animal Breeding and Genetics. 129:474-487. 
  [<a href="http://dx.doi.org/10.1111/jbg.12002">DOI</a>]
  [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23148973">PubMed</a>] 
  [<a href="http://europepmc.org/abstract/MED/23148973">EuropePMC</a>]
  [<a href="http://onlinelibrary.wiley.com/journal/10.1111/%28ISSN%291439-0388/homepage/genomic_selection_virtual_issue.htm">Genomic Selection Virtual Issue</a>]
</li>
</ul>
</p>


<h2>2011</h2>
<p>
<ul>
<li>
Bueno Filho* JS, <strong><u>Morota G</u></strong>*, Tran Q, Maenner MJ, Vera-Cala LM , Engelman CD, and Meyers KJ. 2011.  
Analysis of human mini-exome sequencing data from Genetic Analysis Workshop 17 using a Bayesian hierarchical mixture model. 
BMC Proceedings, 5(Suppl 9):S93. *equal contribution. 
[<a href="http://dx.doi.org/10.1186/1753-6561-5-S9-S93">DOI</a>]
[<a href="http://www.ncbi.nlm.nih.gov/pubmed/22373180">PubMed</a>]
[<a href="http://europepmc.org/abstract/MED/22373180">EuropePMC</a>]
</li>
</ul>
</p>





<h2>bioRxived manuscripts</h2>
<p>
  <ul>
   
    <li>
      Vidigal PMP, Momen M, Costa PMA, Barbosa MHP, <strong><u>Morota G</u></strong>, and Peternelli LA. Regional genomic heritability mapping for agronomic traits in sugarcane. bioRxiv.
      [<a href="https://doi.org/10.1101/2020.04.16.045310">bioRxiv</a>]
    </li>  
    
    <li>
      Campbell MT, Yu H, Momen M, and <strong><u>Morota G</u></strong>. Examining the relationships between phenotypic plasticity and local environments with genomic structural equation models. bioRxiv.
      [<a href="https://doi.org/10.1101/2019.12.11.873257">bioRxiv</a>]
    </li>
    
  
  </ul>
</p>








</body>
</html>
